diff --git a/resource/clientscheme.res.sensitive b/resource/clientscheme.res.sensitive
index 8430470..dffe537 100644
--- a/resource/clientscheme.res.sensitive
+++ b/resource/clientscheme.res.sensitive
@@ -1,23 +1,148 @@
-//
-// TRACKER SCHEME RESOURCE FILE
-//
-// sections:
-//		colors			- all the colors used by the scheme
-//		basesettings	- contains settings for app to use to draw controls
-//		fonts			- list of all the fonts used by app
-//		borders			- description of all the borders
-//
-//
-Scheme
-{
-	//Name - currently overriden in code
-	//{
-	//	"Name"	"ClientScheme"
-	//}
-
-	//////////////////////// COLORS ///////////////////////////
-	Colors
-	{
+Scheme{Colors{
+
+///////////////////////////////////////////////////////////
+////////// BEGIN OF COLOR CUSTOMIZATION SECTION ///////////
+///////////////////////////////////////////////////////////
+
+
+	"HP"					"255 255 255 255"
+	"low HP"				"255 255 255 255"
+	"buffed HP"				"255 255 255 255"
+	"buff shadow"			"0 0 0 255" // "50 205 50 255" to enable the green buff shadow
+	
+	"TargetID low HP"		"255 165 0 255"
+	"TargetID buffed HP"	"50 205 50 255"
+	"TargetID info"			"255 255 0 255"
+
+	"Ammo in Clip"			"255 255 255 255"
+	"Ammo in Reserve"		"168 168 168 255"
+	
+	"Damage Numbers"		"255 255 255 255"
+	"Heal Numbers"			"50 205 50 255"
+	"Last Damage done"		"255 255 0 255"
+	
+	"Ubercharge Bar"		"255 255 255 255"
+	"Ubercharge Full 1"		"61 202 53 255"
+	"Ubercharge Full 2"		"19 165 12 255"
+	
+	"Ubercharge Label"		"255 255 255 255"
+	"Ubercharge Label Full"	"255 255 255 255"
+	
+	"Low-Ammo Clip"			"255 255 255 255"
+	"Low-Ammo Reserve"		"168 168 168 255"
+	"Low-Ammo Shadow"		"255 0 0 255" // "0 0 0 255" to remove the red ammo warning shadow
+
+
+///////////////////////////////////////////////////////////
+/////////// END OF COLOR CUSTOMIZATION SECTION ////////////
+///////////////////////////////////////////////////////////
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+		"G_ToonHUD"				"151 51 51 255" // ToonHUD color
+		"G_White"				"255 255 255 255"
+		"G_Black"				"0 0 0 255"
+		"G_Red"					"230 0 0 255"
+		"G_Shadow"				"0 0 0 220" // Shadow color
+		"G_LightShadow"			"0 0 0 110"
+		"G_Dying"				"212 0 0 230"
+		"G_Heal"				"127 224 74 255" // Heal color
+		"G_Hit"					"235 220 50 255" // Hit color
+		"G_Brown"				"49 44 41 255"
+		"G_Yellow"				"235 220 50 255"
+		"G_HealthBorder"		"25 25 25 255"
+		"G_WhiteShadow"			"255 255 255 50"
+		"G_WhiteShadowDim"		"255 255 255 40"
+		"G_WhiteShadowCharge"	"255 255 255 100"
+		"G_Grey"				"54 54 54 255"
+		"G_LightGrey"			"180 180 180 255"
+		"G_HealthBarBg"			"0 0 0 220"
+		"G_CurrencyBad"			"255 60 60 255"
+		"G_TransparentMenuBG"	"0 0 0 220" // Transparent menu color
+		"G_PanelBg"				"0 0 0 220" // Panel color
+		"G_Shine"				"255 255 255 10" // Shine color
+		"G_Warning"				"255 0 0 255" // Warning color
+		"G_HealthValue"			"255 255 255 255" // Health value color
+		"G_Overheal"			"255 255 255 255" // Overhealed health value color
+		"G_LowHealthValue"		"255 255 255 255" // Low health value color
+		"G_AmmoClipValue"		"255 255 255 255" // Ammo in clip value color
+		"G_AmmoReserveValue"	"255 255 255 255" // Ammo in reserve value color
+		"G_MetalValue"			"255 255 255 255" // Metal color
+		"G_MetalIcon"			"255 255 255 255" // Metal icon color
+		"G_HeadsValue"			"235 220 50 255" // Heads value color
+		"G_HeadsLabel"			"255 255 255 255" // Heads value icon color
+		"G_KillstreakValue"		"255 255 255 255" // Killstreak value color
+		"G_KillstreakIcon"		"255 255 255 255" // Killstreak icon color
+		"G_CrosshairColor"				"255 255 255 255" // Crosshair color
+		"G_CrosshairShadowColor"		"0 0 0 140"	// Crosshair shadow color
+		"G_CrosshairColorLowHealth"		"255 40 40 255" // Crosshair color when low health
+		"G_HitmarkerColor"				"255 0 0 150" // Hitmarker color
+		"G_StickyColor"					"255 255 255 255" // Stickybomb color
+		"G_StickyIconColor"				"255 255 255 255" // Stickybomb icon color
+		"G_CustomTargetID" 				"0 0 0 100" // Custom target id bg color
+		"G_Killfeed_Base"		"0 0 0 120" // Killfeed Base color
+		"G_Killfeed_Local"		"235 235 235 240" // Killfeed Local color
+		"G_CenteredMetalBg"		"50 50 50 120" // Centered metal background color
+		"G_CenteredUberchargeBg" "0 0 0 100" // Centered ubercharge meter background color
+		"G_BrownBg"				"46 43 42 220"
+		"G_DestroyBg"			"120 0 0 150"
+		"G_ScoreboardBlue"		"55 73 85 100"
+		"G_ScoreboardRed"		"106 34 34 100"
+		"G_TeamBlue"		"91 122 140 255"
+		"G_TeamRed"			"189 58 58 255"
+		"G_TeamButtonBlue"		"91 122 140 100"
+		"G_TeamButtonRed"		"189 58 58 100"
+		"G_TeamButtonBlueArmed"	"111 142 160 255"
+		"G_TeamButtonRedArmed"	"209 78 78 255"		
+		"G_Footer"			"45 41 37 255" // Menu footer color
+		"G_OuterBorder"		"0 0 0 255" // Outer border color
+		"G_InnerBorder"		"0 0 0 50" // Inner border color
+		"G_ItemEffectMeterLineColor"	"120 120 120 130"
+		"G_KillfeedBlue"	"83 155 242 255" // Killfeed blue text
+		"G_KillfeedRed"		"243 82 82 255" // Killfeed red text
+		"G_KillfeedIcon"	"255 255 255 255" // Killfeed icon color
+		"G_CustomHealthValue"	"120 120 120 255"
+		"G_MainMenuOverlay"	"17 196 255 0" // Main Menu overlay color
+		"G_MainMenuButton"	"41 37 38 190" // Main Menu button color
+		"G_MainMenuButtonHover"	"41 37 38 240" // Main Menu hovered button color
+		"G_BackpackItemBg"	"46 43 42 220" // Backpack item bg color
+		"G_LoadoutItemBg"	"42 38 37 255" // Loadout item bg color
+		"G_MenuBg"	"46 43 42 255" // Menu bg color
+		"G_MenuText"	"235 226 202 255" // Menu text color
+		"G_ItemDescriptionPanelBg"	"24 22 20 255" // Item description panel bg
+		"G_ButtonBg"	"117 107 94 255" // Button bg color
+		"G_ButtonHoverBg"	"151 51 51 255" // Button hover bg color
+		"G_ButtonText"	"235 226 202 255" // Button text color
+		"G_ButtonHoverText"	"255 255 255 255" // Button hover text color
+		"G_GhostWhite"	"255 255 255 210"
+
+
+
+
 		// base colors
 		"Orange"			"178 82 22 255"
 		"OrangeDim"			"178 82 22 120"
@@ -443,7 +568,6 @@ Scheme
 	{
 		// UI buttons, custom font, (256x64)
 		"Buttons"		"materials/vgui/fonts/buttons_32.vbf"
-		"ButtonsSC"		"materials/vgui/fonts/buttons_sc.vbf"
 	}
 
 
@@ -453,9 +577,1850 @@ Scheme
 	// describes all the fonts
 	Fonts
 	{
-		// fonts are used in order that they are listed
-		// fonts listed later in the order will only be used if they fulfill a range not already filled
-		// if a font fails to load then the subsequent fonts will replace
+	
+		"G_MenuArrowBig"
+		{
+			"1"
+			{
+				"name"			"ToonHUD Icons"
+				"tall"			"26"
+				"weight"		"400"
+				"additive"		"0"
+				"antialias" 	"1"
+			}
+		}
+		
+		"G_MenuArrowSmall"
+		{
+			"1"
+			{
+				"name"			"ToonHUD Icons"
+				"tall"			"15"
+				"weight"		"400"
+				"additive"		"0"
+				"antialias" 	"1"
+			}
+		}
+		
+		"G_BallBig"
+		{
+			"1"
+			{
+				"name"			"ToonHUD Icons"
+				"tall"			"45"
+				"weight"		"400"
+				"additive"		"0"
+				"antialias" 	"1"
+			}
+		}
+		
+		"G_BallMedium"
+		{
+			"1"
+			{
+				"name"			"ToonHUD Icons"
+				"tall"			"20"
+				"weight"		"400"
+				"additive"		"0"
+				"antialias" 	"1"
+			}
+		}
+		
+		"G_BallSmall"
+		{
+			"1"
+			{
+				"name"			"ToonHUD Icons"
+				"tall"			"15"
+				"weight"		"400"
+				"additive"		"0"
+				"antialias" 	"1"
+			}
+		}
+		
+		"G_BallTiny"
+		{
+			"1"
+			{
+				"name"			"ToonHUD Icons"
+				"tall"			"10"
+				"weight"		"400"
+				"additive"		"0"
+				"antialias" 	"1"
+			}
+		}
+		
+		"G_Slash"
+		{
+			"1"
+			{
+				"name"			"ToonHUD Icons"
+				"tall"			"23" [$WINDOWS]
+				"tall"			"37" [!$WINDOWS]
+				"weight"		"400"
+				"additive"		"0"
+				"antialias" 	"1"
+			}
+		}
+		
+		"G_SlashSmall"
+		{
+			"1"
+			{
+				"name"			"ToonHUD Icons"
+				"tall"			"19" [$WINDOWS]
+				"tall"			"30" [!$WINDOWS]
+				"weight"		"400"
+				"additive"		"0"
+				"antialias" 	"1"
+			}
+		}
+		
+		"G_SearchIcon"
+		{
+			"1"
+			{
+				"name"			"ToonHUD Icons"
+				"tall"			"20" [$WINDOWS]
+				"tall"			"25" [!$WINDOWS]
+				"weight"		"400"
+				"additive"		"0"
+				"antialias" 	"1"
+			}
+		}
+		
+		"G_HomeIcon"
+		{
+			"1"
+			{
+				"name"			"ToonHUD Icons"
+				"tall"			"8" [$WINDOWS]
+				"tall"			"10" [!$WINDOWS]
+				"weight"		"400"
+				"additive"		"0"
+				"antialias" 	"1"
+			}
+		}
+		
+		"G_HeadsIcon"
+		{
+			"1"
+			{
+				"name"			"ToonHUD Icons"
+				"tall"			"12" [$WINDOWS]
+				"tall"			"18" [!$WINDOWS]
+				"weight"		"400"
+				"additive"		"0"
+				"antialias" 	"1"
+			}
+		}
+		
+		"G_Stickys"
+		{
+			"1"
+			{
+				"name"			"ToonHUD Icons"
+				"tall"			"60"
+				"weight"		"400"
+				"additive"		"0"
+				"antialias" 	"1"
+			}
+		}
+		
+		"G_Crosshair"
+		{
+			"1"
+			{
+				"name"			"ToonHUD Crosshairs"
+				"tall"			"24"	// Crosshair Size
+				"weight"		"400"
+				"additive"		"0"
+				"antialias" 	"1"
+			}
+		}
+
+		"G_CrosshairShadow"
+		{
+			"1"
+			{
+				"name"			"ToonHUD Crosshairs"
+				"tall"			"24"	// Crosshair Shadow Size
+				"weight"		"400"
+				"additive"		"0"
+				"antialias" 	"1"
+				"blur"			"2" // Crosshair Shadow Blur
+			}
+		}
+		
+		"G_CrosshairShadowSharp"
+		{
+			"1"
+			{
+				"name"			"ToonHUD Crosshairs"
+				"tall"			"24"	// Crosshair Shadow Size
+				"weight"		"400"
+				"additive"		"0"
+				"antialias" 	"1"
+				"blur"			"0"
+			}
+		}
+		
+		"G_Hitmarker"
+		{
+			"1"
+			{
+				"name"			"ToonHUD Hitmarkers"
+				"tall"			"24"	// Hitmarker Size
+				"weight"		"400"
+				"additive"		"0"
+				"antialias" 	"1"
+			}
+		}
+
+		
+		"G_FontHuge"
+		{
+			"1"
+			{
+				"name"			"Komika Axis" [$WINDOWS] // Primary font
+				"name"			"TF2 Build" [!$WINDOWS]
+				"tall"			"60" // Size Huge Primary
+				"weight"		"0"
+				"additive"		"0"
+				"antialias"		"1"
+			}
+		}
+		
+		"G_FontBig"
+		{
+			"1"
+			{
+				"name"			"Komika Axis" [$WINDOWS] // Primary font
+				"name"			"TF2 Build" [!$WINDOWS]
+				"tall"			"40" // Size Big Primary
+				"weight"		"0"
+				"additive"		"0"
+				"antialias"		"1"
+			}
+		}
+		
+		"G_FontMedium"
+		{
+			"1"
+			{
+				"name"			"Komika Axis" [$WINDOWS] // Primary font
+				"name"			"TF2 Build" [!$WINDOWS]
+				"tall"			"25" // Size Medium Primary
+				"weight"		"0"
+				"additive"		"0"
+				"antialias"		"1"
+			}
+		}
+		
+		"G_FontSmall"
+		{
+			"1"
+			{
+				"name"			"Komika Axis" [$WINDOWS] // Primary font
+				"name"			"TF2 Build" [!$WINDOWS]
+				"tall"			"15" // Size Small Primary
+				"weight"		"0"
+				"additive"		"0"
+				"antialias" 	"1"
+			}
+		}
+		
+		"G_FontTiny"
+		{
+			"1"
+			{
+				"name"			"Komika Axis" [$WINDOWS] // Primary font
+				"name"			"TF2 Build" [!$WINDOWS]
+				"tall"			"14" // Size Tiny Primary
+				"weight"		"0"
+				"additive"		"0"
+				"antialias" 	"1"
+			}
+		}
+		
+		
+		"G_FontHuge_2"
+		{
+			"1"
+			{
+				"name"			"Dimbo" [$WINDOWS] // Secondary font
+				"name"			"TF2 Secondary" [!$WINDOWS]
+				"tall"			"25" // Size Huge Secondary
+				"weight"		"0"
+				"additive"		"0"
+				"antialias" 	"1"
+			}
+		}
+		
+		"G_FontBig_2"
+		{
+			"1"
+			{
+				"name"			"Dimbo" [$WINDOWS] // Secondary font
+				"name"			"TF2 Secondary" [!$WINDOWS]
+				"tall"			"19" // Size Big Secondary
+				"weight"		"0"
+				"additive"		"0"
+				"antialias" 	"1"
+			}
+		}
+		
+		"G_FontMedium_2"
+		{
+			"1"
+			{
+				"name"			"Dimbo" [$WINDOWS] // Secondary font
+				"name"			"TF2 Secondary" [!$WINDOWS]
+				"tall"			"16" // Size Medium Secondary
+				"weight"		"0"
+				"additive"		"0"
+				"antialias" 	"1"
+			}
+		}
+		
+		"G_FontSmall_2"
+		{
+			"1"
+			{
+				"name"			"Dimbo" [$WINDOWS] // Secondary font
+				"name"			"TF2 Secondary" [!$WINDOWS]
+				"tall"			"12" // Size Small Secondary
+				"weight"		"0"
+				"additive"		"0"
+				"antialias" 	"1"
+			}
+		}
+		
+		"G_FontTiny_2"
+		{
+			"1"
+			{
+				"name"			"Dimbo" [$WINDOWS] // Secondary font
+				"name"			"TF2 Secondary" [!$WINDOWS]
+				"tall"			"10" // Size Tiny Secondary
+				"weight"		"0"
+				"additive"		"0"
+				"antialias" 	"1"
+				"dropshadow"	"0"
+			}
+		}
+		
+		"G_FontDamage"
+		{
+			"1"
+			{
+				"name"			"Komika Axis" [$WINDOWS] // Damage font
+				"name"			"TF2 Build" [!$WINDOWS]
+				"tall"			"20" // Damage size
+				"weight"		"0"
+				"additive"		"0"
+				"antialias"		"1"
+				"outline"		"1" // Damage outline
+			}
+		}
+		
+		"G_FontCritDamage"
+		{
+			"1"
+			{
+				"name"			"Komika Axis" [$WINDOWS] // Crit damage font
+				"name"			"TF2 Build" [!$WINDOWS]
+				"tall"			"22" // Crit damage size
+				"weight"		"0"
+				"additive"		"0"
+				"antialias"		"1"
+				"outline"		"1" // Crit damage outline
+			}
+		}
+		
+		"G_FontSourceScheme"
+		{
+			"1"
+			{
+				"name"		"Verdana"
+				"tall"		"14"
+				"weight"	"400"
+			}
+		}
+		
+		"G_FontNumberButton"
+		{
+			"1"
+			{
+				"name"			"Verdana"
+				"tall"			"9"
+				"weight"		"500"
+				"antialias" 	"1"
+			}
+		}
+		
+		"G_FontHugeBlur"
+		{
+			"1"
+			{
+				"name"			"Komika Axis" [$WINDOWS] // Primary font
+				"name"			"TF2 Build" [!$WINDOWS]
+				"tall"			"60" // Size Huge Primary
+				"weight"		"0"
+				"additive"		"0"
+				"antialias"		"1"
+				"blur"			"3"
+			}
+		}
+		
+		"G_FontBigBlur"
+		{
+			"1"
+			{
+				"name"			"Komika Axis" [$WINDOWS] // Primary font
+				"name"			"TF2 Build" [!$WINDOWS]
+				"tall"			"40" // Size Big Primary
+				"weight"		"0"
+				"additive"		"0"
+				"antialias"		"1"
+				"blur"			"3"
+			}
+		}
+		
+		"G_FontMediumBlur"
+		{
+			"1"
+			{
+				"name"			"Komika Axis" [$WINDOWS] // Primary font
+				"name"			"TF2 Build" [!$WINDOWS]
+				"tall"			"25" // Size Medium Primary
+				"weight"		"0"
+				"additive"		"0"
+				"antialias"		"1"
+				"blur"			"3"
+			}
+		}
+		
+		"G_FontSmallBlur"
+		{
+			"1"
+			{
+				"name"			"Komika Axis" [$WINDOWS] // Primary font
+				"name"			"TF2 Build" [!$WINDOWS]
+				"tall"			"15" // Size Small Primary
+				"weight"		"0"
+				"additive"		"0"
+				"antialias" 	"1"
+				"blur"			"3"
+			}
+		}
+		
+		"G_FontTinyBlur"
+		{
+			"1"
+			{
+				"name"			"Komika Axis" [$WINDOWS] // Primary font
+				"name"			"TF2 Build" [!$WINDOWS]
+				"tall"			"14" // Size Tiny Primary
+				"weight"		"0"
+				"additive"		"0"
+				"antialias" 	"1"
+				"blur"			"3"
+			}
+		}
+		
+		"G_Empty"
+		{
+			"1"
+			{
+				"name"			"Verdana"
+				"tall"			"1"
+				"weight"		"0"
+				"additive"		"0"
+				"antialias" 	"0"
+			}
+		}
+		
+		"xHairDmg"
+		{
+			"1"
+			{
+				"name" "Crosshairs"
+				"tall" "28"
+				"weight" "0"
+				"antialias" "1"
+			}
+		}
+		
+		surface8
+		{
+			"1"
+			{
+				"name"		"Surface-Medium"
+				"tall"		"8"
+				"additive"	"0"
+				"antialias" 	"1"
+				"outline"	"0"
+			}
+		}
+		surface9
+		{
+			"1"
+			{
+				"name"		"Surface-Medium"
+				"tall"		"9"
+				"additive"	"0"
+				"antialias" 	"1"
+				"outline"	"0"
+			}
+		}
+		surface10
+		{
+			"1"
+			{
+				"name"		"Surface-Medium"
+				"tall"		"10"
+				"additive"	"0"
+				"antialias" 	"1"
+				"outline"	"0"
+			}
+		}
+		surface11
+		{
+			"1"
+			{
+				"name"		"Surface-Medium"
+				"tall"		"11"
+				"additive"	"0"
+				"antialias" 	"1"
+				"outline"	"0"
+			}
+		}
+		surface12
+		{
+			"1"
+			{
+				"name"		"Surface-Medium"
+				"tall"		"12"
+				"additive"	"0"
+				"antialias" 	"1"
+				"outline"	"0"
+			}
+		}
+		surface13
+		{
+			"1"
+			{
+				"name"		"Surface-Medium"
+				"tall"		"13"
+				"additive"	"0"
+				"antialias" 	"1"
+				"outline"	"0"
+			}
+		}
+		surface14
+		{
+			"1"
+			{
+				"name"		"Surface-Medium"
+				"tall"		"14"
+				"additive"	"0"
+				"antialias" 	"1"
+				"outline"	"0"
+			}
+		}
+		surface14shadow
+		{
+			"1"
+			{
+				"name"		"Surface-Medium"
+				"tall"		"14"
+				"additive"	"0"
+				"antialias" 	"1"
+				"outline"	"0"
+				"dropshadow"	"1"
+			}
+		}
+		surface15
+		{
+			"1"
+			{
+				"name"		"Surface-Medium"
+				"tall"		"15"
+				"additive"	"0"
+				"antialias" 	"1"
+				"outline"	"0"
+			}
+		}
+		surface15shadow
+		{
+			"1"
+			{
+				"name"		"Surface-Medium"
+				"tall"		"15"
+				"additive"	"0"
+				"antialias" 	"1"
+				"outline"	"0"
+				"dropshadow"	"1"
+			}
+		}
+		surface16
+		{
+			"1"
+			{
+				"name"		"Surface-Medium"
+				"tall"		"16"
+				"additive"	"0"
+				"antialias" 	"1"
+				"outline"	"0"
+			}
+		}
+		surface18
+		{
+			"1"
+			{
+				"name"		"Surface-Medium"
+				"tall"		"18"
+				"additive"	"0"
+				"antialias" 	"1"
+				"outline"	"0"
+			}
+		}
+		surface18outline
+		{
+			"1"
+			{
+				"name"		"Surface-Medium"
+				"tall"		"18"
+				"additive"	"0"
+				"antialias" 	"1"
+				"outline"	"1"
+			}
+		}
+		surface20
+		{
+			"1"
+			{
+				"name"		"Surface-Medium"
+				"tall"		"20"
+				"additive"	"0"
+				"antialias" 	"1"
+				"outline"	"0"
+			}
+		}
+		surface20outline
+		{
+			"1"
+			{
+				"name"		"Surface-Medium"
+				"tall"		"20"
+				"additive"	"0"
+				"antialias" 	"1"
+				"outline"	"1"
+			}
+		}
+		surface23
+		{
+			"1"
+			{
+				"name"		"Surface-Medium"
+				"tall"		"23"
+				"additive"	"0"
+				"antialias" 	"1"
+				"outline"	"0"
+			}
+		}
+		surface23outline
+		{
+			"1"
+			{
+				"name"		"Surface-Medium"
+				"tall"		"23"
+				"additive"	"0"
+				"antialias" 	"1"
+				"outline"	"1"
+			}
+		}
+		surface24
+		{
+			"1"
+			{
+				"name"		"SurfaceSharpened-Medium"
+				"tall"		"24"
+				"additive"	"0"
+				"antialias" 	"1"
+				"outline"	"0"
+			}
+		}
+		surface24shadow
+		{
+			"1"
+			{
+				"name"		"SurfaceSharpened-Medium"
+				"tall"		"24"
+				"additive"	"0"
+				"antialias" 	"1"
+				"outline"	"0"
+				"dropshadow"	"1"
+			}
+		}
+		surface26
+		{
+			"1"
+			{
+				"name"		"SurfaceSharpened-Medium"
+				"tall"		"26"
+				"additive"	"0"
+				"antialias" 	"1"
+				"outline"	"0"
+			}
+		}
+		surface28
+		{
+			"1"
+			{
+				"name"		"SurfaceSharpened-Medium"
+				"tall"		"28"
+				"additive"	"0"
+				"antialias" 	"1"
+				"outline"	"0"
+			}
+		}
+		surface32
+		{
+			"1"
+			{
+				"name"		"SurfaceSharpened-Medium"
+				"tall"		"32"
+				"additive"	"0"
+				"antialias" 	"1"
+				"outline"	"0"
+			}
+		}
+		surface36
+		{
+			"1"
+			{
+				"name"		"SurfaceSharpened-Medium"
+				"tall"		"36"
+				"additive"	"0"
+				"antialias" 	"1"
+				"outline"	"0"
+			}
+		}
+		surface48
+		{
+			"1"
+			{
+				"name"		"SurfaceSharpened-Medium"
+				"tall"		"48"
+				"additive"	"0"
+				"antialias" 	"1"
+				"outline"	"0"
+			}
+		}
+		surface60
+		{
+			"1"
+			{
+				"name"		"SurfaceSharpened-Medium"
+				"tall"		"60"
+				"additive"	"0"
+				"antialias" 	"1"
+				"outline"	"0"
+			}
+		}
+		surfaceUber
+		{
+			"1"
+			{
+				"name"		"SurfaceSharpened-Medium"
+				"tall"		"60"
+				"additive"	"0"
+				"antialias" 	"1"
+				"outline"	"0"
+				"dropshadow"	"1"
+			}
+		}
+		surfaceNumbers14
+		{
+			"1"
+			{
+				"name"		"SurfaceNumbers-Medium"
+				"tall"		"14"
+				"additive"	"0"
+				"antialias" 	"1"
+				"outline"	"0"
+			}
+		}
+		surfaceNumbers16
+		{
+			"1"
+			{
+				"name"		"SurfaceNumbers-Medium"
+				"tall"		"16"
+				"additive"	"0"
+				"antialias" 	"1"
+				"outline"	"0"
+			}
+		}
+		surfaceNumbers18
+		{
+			"1"
+			{
+				"name"		"SurfaceNumbers-Medium"
+				"tall"		"18"
+				"additive"	"0"
+				"antialias" 	"1"
+				"outline"	"0"
+			}
+		}
+		surfaceNumbers28
+		{
+			"1"
+			{
+				"name"		"SurfaceNumbers-Medium"
+				"tall"		"28"
+				"additive"	"0"
+				"antialias" 	"1"
+				"outline"	"0"
+			}
+		}
+		ItemEffectMeterLabels
+		{
+			"1"
+			{
+				"name"		"Verdana"
+				"tall"		"8"
+				"additive"	"0"
+				"antialias" 	"1"
+				"outline"	"0"
+			}
+		}	
+		Blank
+		{
+			"1"
+			{
+				"name"		"Blank"
+				"tall"		"8"
+				"additive"	"0"
+				"antialias" 	"1"
+				"outline"	"0"
+			}
+		}
+		"Blocks"
+		{
+			"1"
+			{
+				"name"		"Blocks"
+				"tall"		"72"
+				"additive"	"0"
+				"antialias" "0"
+			}
+		}
+		"size: 10, outline: off"
+		{
+			"1"
+			{
+				"name"	"broeselhud Crosshairs"
+				"tall"	"10"
+				"antialias" "1"
+				"additive"	"0"
+				"outline"	"0"
+			}
+		}
+		"size: 11, outline: off"
+		{
+			"1"
+			{
+				"name"	"broeselhud Crosshairs"
+				"tall"	"11"
+				"antialias" "1"
+				"additive"	"0"
+				"outline"	"0"
+			}
+		}
+		"size: 12, outline: off"
+		{
+			"1"
+			{
+				"name"	"broeselhud Crosshairs"
+				"tall"	"12"
+				"antialias" "1"
+				"additive"	"0"
+				"outline"	"0"
+			}
+		}
+		"size: 13, outline: off"
+		{
+			"1"
+			{
+				"name"	"broeselhud Crosshairs"
+				"tall"	"13"
+				"antialias" "1"
+				"additive"	"0"
+				"outline"	"0"
+			}
+		}
+		"size: 14, outline: off"
+		{
+			"1"
+			{
+				"name"	"broeselhud Crosshairs"
+				"tall"	"14"
+				"antialias" "1"
+				"additive"	"0"
+				"outline"	"0"
+			}
+		}
+		"size: 15, outline: off"
+		{
+			"1"
+			{
+				"name"	"broeselhud Crosshairs"
+				"tall"	"15"
+				"antialias" "1"
+				"additive"	"0"
+				"outline"	"0"
+			}
+		}
+		"size: 16, outline: off"
+		{
+			"1"
+			{
+				"name"	"broeselhud Crosshairs"
+				"tall"	"16"
+				"antialias" "1"
+				"additive"	"0"
+				"outline"	"0"
+			}
+		}
+		"size: 17, outline: off"
+		{
+			"1"
+			{
+				"name"	"broeselhud Crosshairs"
+				"tall"	"17"
+				"antialias" "1"
+				"additive"	"0"
+				"outline"	"0"
+			}
+		}
+		"size: 18, outline: off"
+		{
+			"1"
+			{
+				"name"	"broeselhud Crosshairs"
+				"tall"	"18"
+				"antialias" "1"
+				"additive"	"0"
+				"outline"	"0"
+			}
+		}
+		"size: 19, outline: off"
+		{
+			"1"
+			{
+				"name"	"broeselhud Crosshairs"
+				"tall"	"19"
+				"antialias" "1"
+				"additive"	"0"
+				"outline"	"0"
+			}
+		}
+		"size: 20, outline: off"
+		{
+			"1"
+			{
+				"name"	"broeselhud Crosshairs"
+				"tall"	"20"
+				"antialias" "1"
+				"additive"	"0"
+				"outline"	"0"
+			}
+		}
+		"size: 21, outline: off"
+		{
+			"1"
+			{
+				"name"	"broeselhud Crosshairs"
+				"tall"	"21"
+				"antialias" "1"
+				"additive"	"0"
+				"outline"	"0"
+			}
+		}
+		"size: 22, outline: off"
+		{
+			"1"
+			{
+				"name"	"broeselhud Crosshairs"
+				"tall"	"22"
+				"antialias" "1"
+				"additive"	"0"
+				"outline"	"0"
+			}
+		}
+		"size: 23, outline: off"
+		{
+			"1"
+			{
+				"name"	"broeselhud Crosshairs"
+				"tall"	"23"
+				"antialias" "1"
+				"additive"	"0"
+				"outline"	"0"
+			}
+		}
+		"size: 24, outline: off"
+		{
+			"1"
+			{
+				"name"	"broeselhud Crosshairs"
+				"tall"	"24"
+				"antialias" "1"
+				"additive"	"0"
+				"outline"	"0"
+			}
+		}
+		"size: 25, outline: off"
+		{
+			"1"
+			{
+				"name"	"broeselhud Crosshairs"
+				"tall"	"25"
+				"antialias" "1"
+				"additive"	"0"
+				"outline"	"0"
+			}
+		}
+		"size: 26, outline: off"
+		{
+			"1"
+			{
+				"name"	"broeselhud Crosshairs"
+				"tall"	"26"
+				"antialias" "1"
+				"additive"	"0"
+				"outline"	"0"
+			}
+		}
+		"size: 27, outline: off"
+		{
+			"1"
+			{
+				"name"	"broeselhud Crosshairs"
+				"tall"	"26"
+				"antialias" "1"
+				"additive"	"0"
+				"outline"	"0"
+			}
+		}
+		"size: 28, outline: off"
+		{
+			"1"
+			{
+				"name"	"broeselhud Crosshairs"
+				"tall"	"28"
+				"antialias" "1"
+				"additive"	"0"
+				"outline"	"0"
+			}
+		}
+		"size: 29, outline: off"
+		{
+			"1"
+			{
+				"name"	"broeselhud Crosshairs"
+				"tall"	"29"
+				"antialias" "1"
+				"additive"	"0"
+				"outline"	"0"
+			}
+		}
+		"size: 30, outline: off"
+		{
+			"1"
+			{
+				"name"	"broeselhud Crosshairs"
+				"tall"	"30"
+				"antialias" "1"
+				"additive"	"0"
+				"outline"	"0"
+			}
+		}
+		"size: 31, outline: off"
+		{
+			"1"
+			{
+				"name"	"broeselhud Crosshairs"
+				"tall"	"31"
+				"antialias" "1"
+				"additive"	"0"
+				"outline"	"0"
+			}
+		}
+		"size: 32, outline: off"
+		{
+			"1"
+			{
+				"name"	"broeselhud Crosshairs"
+				"tall"	"32"
+				"antialias" "1"
+				"additive"	"0"
+				"outline"	"0"
+			}
+		}
+		"size: 33, outline: off"
+		{
+			"1"
+			{
+				"name"	"broeselhud Crosshairs"
+				"tall"	"33"
+				"antialias" "1"
+				"additive"	"0"
+				"outline"	"0"
+			}
+		}
+		"size: 34, outline: off"
+		{
+			"1"
+			{
+				"name"	"broeselhud Crosshairs"
+				"tall"	"34"
+				"antialias" "1"
+				"additive"	"0"
+				"outline"	"0"
+			}
+		}
+		"size: 35, outline: off"
+		{
+			"1"
+			{
+				"name"	"broeselhud Crosshairs"
+				"tall"	"35"
+				"antialias" "1"
+				"additive"	"0"
+				"outline"	"0"
+			}
+		}
+		"size: 36, outline: off"
+		{
+			"1"
+			{
+				"name"	"broeselhud Crosshairs"
+				"tall"	"36"
+				"antialias" "1"
+				"additive"	"0"
+				"outline"	"0"
+			}
+		}
+		"size: 37, outline: off"
+		{
+			"1"
+			{
+				"name"	"broeselhud Crosshairs"
+				"tall"	"37"
+				"antialias" "1"
+				"additive"	"0"
+				"outline"	"0"
+			}
+		}
+		"size: 38, outline: off"
+		{
+			"1"
+			{
+				"name"	"broeselhud Crosshairs"
+				"tall"	"38"
+				"antialias" "1"
+				"additive"	"0"
+				"outline"	"0"
+			}
+		}
+		"size: 39, outline: off"
+		{
+			"1"
+			{
+				"name"	"broeselhud Crosshairs"
+				"tall"	"39"
+				"antialias" "1"
+				"additive"	"0"
+				"outline"	"0"
+			}
+		}
+		"size: 40, outline: off"
+		{
+			"1"
+			{
+				"name"	"broeselhud Crosshairs"
+				"tall"	"40"
+				"antialias" "1"
+				"additive"	"0"
+				"outline"	"0"
+			}
+		}
+		"size: 41, outline: off"
+		{
+			"1"
+			{
+				"name"	"broeselhud Crosshairs"
+				"tall"	"41"
+				"antialias" "1"
+				"additive"	"0"
+				"outline"	"0"
+			}
+		}
+		"size: 42, outline: off"
+		{
+			"1"
+			{
+				"name"	"broeselhud Crosshairs"
+				"tall"	"42"
+				"antialias" "1"
+				"additive"	"0"
+				"outline"	"0"
+			}
+		}
+		"size: 43, outline: off"
+		{
+			"1"
+			{
+				"name"	"broeselhud Crosshairs"
+				"tall"	"43"
+				"antialias" "1"
+				"additive"	"0"
+				"outline"	"0"
+			}
+		}
+		"size: 44, outline: off"
+		{
+			"1"
+			{
+				"name"	"broeselhud Crosshairs"
+				"tall"	"44"
+				"antialias" "1"
+				"additive"	"0"
+				"outline"	"0"
+			}
+		}
+		"size: 45, outline: off"
+		{
+			"1"
+			{
+				"name"	"broeselhud Crosshairs"
+				"tall"	"45"
+				"antialias" "1"
+				"additive"	"0"
+				"outline"	"0"
+			}
+		}
+		"size: 46, outline: off"
+		{
+			"1"
+			{
+				"name"	"broeselhud Crosshairs"
+				"tall"	"46"
+				"antialias" "1"
+				"additive"	"0"
+				"outline"	"0"
+			}
+		}
+		"size: 47, outline: off"
+		{
+			"1"
+			{
+				"name"	"broeselhud Crosshairs"
+				"tall"	"47"
+				"antialias" "1"
+				"additive"	"0"
+				"outline"	"0"
+			}
+		}
+		"size: 48, outline: off"
+		{
+			"1"
+			{
+				"name"	"broeselhud Crosshairs"
+				"tall"	"48"
+				"antialias" "1"
+				"additive"	"0"
+				"outline"	"0"
+			}
+		}
+		"size: 49, outline: off"
+		{
+			"1"
+			{
+				"name"	"broeselhud Crosshairs"
+				"tall"	"49"
+				"antialias" "1"
+				"additive"	"0"
+				"outline"	"0"
+			}
+		}
+		"size: 50, outline: off"
+		{
+			"1"
+			{
+				"name"	"broeselhud Crosshairs"
+				"tall"	"50"
+				"antialias" "1"
+				"additive"	"0"
+				"outline"	"0"
+			}
+		}
+		"size: 12, outline: on"
+		{
+			"1"
+			{
+				"name"	"broeselhud Crosshairs"
+				"tall"	"12"
+				"antialias" "0"
+				"additive"	"0"
+				"outline"	"1"
+			}
+		}
+		"size: 13, outline: on"
+		{
+			"1"
+			{
+				"name"	"broeselhud Crosshairs"
+				"tall"	"13"
+				"antialias" "0"
+				"additive"	"0"
+				"outline"	"1"
+			}
+		}
+		"size: 14, outline: on"
+		{
+			"1"
+			{
+				"name"	"broeselhud Crosshairs"
+				"tall"	"14"
+				"antialias" "0"
+				"additive"	"0"
+				"outline"	"1"
+			}
+		}
+		"size: 15, outline: on"
+		{
+			"1"
+			{
+				"name"	"broeselhud Crosshairs"
+				"tall"	"15"
+				"antialias" "0"
+				"additive"	"0"
+				"outline"	"1"
+			}
+		}
+		"size: 16, outline: on"
+		{
+			"1"
+			{
+				"name"	"broeselhud Crosshairs"
+				"tall"	"16"
+				"antialias" "0"
+				"additive"	"0"
+				"outline"	"1"
+			}
+		}
+		"size: 17, outline: on"
+		{
+			"1"
+			{
+				"name"	"broeselhud Crosshairs"
+				"tall"	"17"
+				"antialias" "0"
+				"additive"	"0"
+				"outline"	"1"
+			}
+		}
+		"size: 18, outline: on"
+		{
+			"1"
+			{
+				"name"	"broeselhud Crosshairs"
+				"tall"	"18"
+				"antialias" "0"
+				"additive"	"0"
+				"outline"	"1"
+			}
+		}
+		"size: 19, outline: on"
+		{
+			"1"
+			{
+				"name"	"broeselhud Crosshairs"
+				"tall"	"19"
+				"antialias" "0"
+				"additive"	"0"
+				"outline"	"1"
+			}
+		}
+		"size: 20, outline: on"
+		{
+			"1"
+			{
+				"name"	"broeselhud Crosshairs"
+				"tall"	"20"
+				"antialias" "0"
+				"additive"	"0"
+				"outline"	"1"
+			}
+		}
+		"size: 21, outline: on"
+		{
+			"1"
+			{
+				"name"	"broeselhud Crosshairs"
+				"tall"	"21"
+				"antialias" "0"
+				"additive"	"0"
+				"outline"	"1"
+			}
+		}
+		"size: 22, outline: on"
+		{
+			"1"
+			{
+				"name"	"broeselhud Crosshairs"
+				"tall"	"22"
+				"antialias" "0"
+				"additive"	"0"
+				"outline"	"1"
+			}
+		}
+		"size: 23, outline: on"
+		{
+			"1"
+			{
+				"name"	"broeselhud Crosshairs"
+				"tall"	"23"
+				"antialias" "0"
+				"additive"	"0"
+				"outline"	"1"
+			}
+		}
+		"size: 24, outline: on"
+		{
+			"1"
+			{
+				"name"	"broeselhud Crosshairs"
+				"tall"	"24"
+				"antialias" "0"
+				"additive"	"0"
+				"outline"	"1"
+			}
+		}
+		"size: 25, outline: on"
+		{
+			"1"
+			{
+				"name"	"broeselhud Crosshairs"
+				"tall"	"25"
+				"antialias" "0"
+				"additive"	"0"
+				"outline"	"1"
+			}
+		}
+		"size: 26, outline: on"
+		{
+			"1"
+			{
+				"name"	"broeselhud Crosshairs"
+				"tall"	"26"
+				"antialias" "0"
+				"additive"	"0"
+				"outline"	"1"
+			}
+		}
+		"size: 27, outline: on"
+		{
+			"1"
+			{
+				"name"	"broeselhud Crosshairs"
+				"tall"	"26"
+				"antialias" "0"
+				"additive"	"0"
+				"outline"	"1"
+			}
+		}
+		"size: 28, outline: on"
+		{
+			"1"
+			{
+				"name"	"broeselhud Crosshairs"
+				"tall"	"28"
+				"antialias" "0"
+				"additive"	"0"
+				"outline"	"1"
+			}
+		}
+		"size: 29, outline: on"
+		{
+			"1"
+			{
+				"name"	"broeselhud Crosshairs"
+				"tall"	"29"
+				"antialias" "0"
+				"additive"	"0"
+				"outline"	"1"
+			}
+		}
+		"size: 30, outline: on"
+		{
+			"1"
+			{
+				"name"	"broeselhud Crosshairs"
+				"tall"	"30"
+				"antialias" "0"
+				"additive"	"0"
+				"outline"	"1"
+			}
+		}
+		"size: 31, outline: on"
+		{
+			"1"
+			{
+				"name"	"broeselhud Crosshairs"
+				"tall"	"31"
+				"antialias" "0"
+				"additive"	"0"
+				"outline"	"1"
+			}
+		}
+		"size: 32, outline: on"
+		{
+			"1"
+			{
+				"name"	"broeselhud Crosshairs"
+				"tall"	"32"
+				"antialias" "0"
+				"additive"	"0"
+				"outline"	"1"
+			}
+		}
+		"size: 33, outline: on"
+		{
+			"1"
+			{
+				"name"	"broeselhud Crosshairs"
+				"tall"	"33"
+				"antialias" "0"
+				"additive"	"0"
+				"outline"	"1"
+			}
+		}
+		"size: 34, outline: on"
+		{
+			"1"
+			{
+				"name"	"broeselhud Crosshairs"
+				"tall"	"34"
+				"antialias" "0"
+				"additive"	"0"
+				"outline"	"1"
+			}
+		}
+		"size: 35, outline: on"
+		{
+			"1"
+			{
+				"name"	"broeselhud Crosshairs"
+				"tall"	"35"
+				"antialias" "0"
+				"additive"	"0"
+				"outline"	"1"
+			}
+		}
+		"size: 36, outline: on"
+		{
+			"1"
+			{
+				"name"	"broeselhud Crosshairs"
+				"tall"	"36"
+				"antialias" "0"
+				"additive"	"0"
+				"outline"	"1"
+			}
+		}
+		"size: 37, outline: on"
+		{
+			"1"
+			{
+				"name"	"broeselhud Crosshairs"
+				"tall"	"37"
+				"antialias" "0"
+				"additive"	"0"
+				"outline"	"1"
+			}
+		}
+		"size: 38, outline: on"
+		{
+			"1"
+			{
+				"name"	"broeselhud Crosshairs"
+				"tall"	"38"
+				"antialias" "0"
+				"additive"	"0"
+				"outline"	"1"
+			}
+		}
+		"size: 39, outline: on"
+		{
+			"1"
+			{
+				"name"	"broeselhud Crosshairs"
+				"tall"	"39"
+				"antialias" "0"
+				"additive"	"0"
+				"outline"	"1"
+			}
+		}
+		"size: 40, outline: on"
+		{
+			"1"
+			{
+				"name"	"broeselhud Crosshairs"
+				"tall"	"40"
+				"antialias" "0"
+				"additive"	"0"
+				"outline"	"1"
+			}
+		}
+		"size: 41, outline: on"
+		{
+			"1"
+			{
+				"name"	"broeselhud Crosshairs"
+				"tall"	"41"
+				"antialias" "0"
+				"additive"	"0"
+				"outline"	"1"
+			}
+		}
+		"size: 42, outline: on"
+		{
+			"1"
+			{
+				"name"	"broeselhud Crosshairs"
+				"tall"	"42"
+				"antialias" "0"
+				"additive"	"0"
+				"outline"	"1"
+			}
+		}
+		"size: 43, outline: on"
+		{
+			"1"
+			{
+				"name"	"broeselhud Crosshairs"
+				"tall"	"43"
+				"antialias" "0"
+				"additive"	"0"
+				"outline"	"1"
+			}
+		}
+		"size: 44, outline: on"
+		{
+			"1"
+			{
+				"name"	"broeselhud Crosshairs"
+				"tall"	"44"
+				"antialias" "0"
+				"additive"	"0"
+				"outline"	"1"
+			}
+		}
+		"size: 45, outline: on"
+		{
+			"1"
+			{
+				"name"	"broeselhud Crosshairs"
+				"tall"	"45"
+				"antialias" "0"
+				"additive"	"0"
+				"outline"	"1"
+			}
+		}
+		"size: 46, outline: on"
+		{
+			"1"
+			{
+				"name"	"broeselhud Crosshairs"
+				"tall"	"46"
+				"antialias" "0"
+				"additive"	"0"
+				"outline"	"1"
+			}
+		}
+		"size: 47, outline: on"
+		{
+			"1"
+			{
+				"name"	"broeselhud Crosshairs"
+				"tall"	"47"
+				"antialias" "0"
+				"additive"	"0"
+				"outline"	"1"
+			}
+		}
+		"size: 48, outline: on"
+		{
+			"1"
+			{
+				"name"	"broeselhud Crosshairs"
+				"tall"	"48"
+				"antialias" "0"
+				"additive"	"0"
+				"outline"	"1"
+			}
+		}
+		"size: 49, outline: on"
+		{
+			"1"
+			{
+				"name"	"broeselhud Crosshairs"
+				"tall"	"49"
+				"antialias" "0"
+				"additive"	"0"
+				"outline"	"1"
+			}
+		}
+		"size: 50, outline: on"
+		{
+			"1"
+			{
+				"name"	"broeselhud Crosshairs"
+				"tall"	"50"
+				"antialias" "0"
+				"additive"	"0"
+				"outline"	"1"
+			}
+		}
+		"konr16"
+		{
+			"1"
+			{
+				"name"			"Garm3nFont"
+				"tall"			"16" 
+				"weight"		"0"
+				"antialias"		"1"
+			}
+		}
+		"konr18"
+		{
+			"1"
+			{
+				"name"			"Garm3nFont"
+				"tall"			"18" 
+				"weight"		"0"
+				"antialias"		"1"
+			}
+		}
+		"konr20"
+		{
+			"1"
+			{
+				"name"			"Garm3nFont"
+				"tall"			"20" 
+				"weight"		"0"
+				"antialias"		"1"
+			}
+		}
+		"konr22"
+		{
+			"1"
+			{		
+				"name"			"Garm3nFont"
+				"tall"			"22" 
+				"weight"		"0"
+				"antialias"		"1"
+			}
+		}
+		"konr24"
+		{
+			"1"
+			{		
+				"name"			"Garm3nFont"
+				"tall"			"24" 
+				"weight"		"0"
+				"antialias"		"1"
+			}
+		}
+		"seeker10"
+		{
+			"1"
+			{
+				"name"			"tf2cross"
+				"tall"			"10"
+				"weight"		"0"
+				"antialias"		"1"
+				"additive"		"0"
+				"outline"		"0"
+			}
+		}
+		"seeker10outline"
+		{
+			"1"
+			{
+				"name"			"tf2cross"
+				"tall"			"10"
+				"weight"		"0"
+				"antialias"		"1"
+				"additive"		"0"
+				"outline"		"1"
+			}
+		}
+		"seeker11"
+		{
+			"1"
+			{
+				"name"			"tf2cross"
+				"tall"			"11"
+				"weight"		"0"
+				"antialias"		"1"
+				"additive"		"0"
+				"outline"		"0"
+			}
+		}
+		"seeker11outline"
+		{
+			"1"
+			{
+				"name"			"tf2cross"
+				"tall"			"11"
+				"weight"		"0"
+				"antialias"		"1"
+				"additive"		"0"
+				"outline"		"1"
+			}
+		}
+		"seeker12"
+		{
+			"1"
+			{
+				"name"			"tf2cross"
+				"tall"			"12"
+				"weight"		"0"
+				"antialias"		"1"
+				"additive"		"0"
+				"outline"		"0"
+			}
+		}
+		"seeker12outline"
+		{
+			"1"
+			{
+				"name"			"tf2cross"
+				"tall"			"12"
+				"weight"		"0"
+				"antialias"		"1"
+				"additive"		"0"
+				"outline"		"1"
+			}
+		}
+		
 		"Default"
 		{
 			"1"
@@ -1639,7 +3604,7 @@ Scheme
 			"1"
 			{
 				"name"		"TF2 Secondary"
-				"tall"		"20"
+				"tall"		"24"
 				"weight"	"500"
 				"range"		"0x0000 0x007F"	//	Basic Latin
 				"antialias" "1"
@@ -1723,51 +3688,7 @@ Scheme
 				"antialias" "1"
 			}
 		}
-		"MatchSummaryTeamScores"
-		{
-			"1"
-			{
-				"name"		"TF2"
-				"tall"		"36"
-				"weight"	"500"
-				"range"		"0x0000 0x007F"	//	Basic Latin
-				"antialias" "1"
-				"additive"	"0"
-			}
-		}
-		"MatchSummaryStatsAndMedals"
-		{
-			"1"
-			{
-				"name"		"TF2 Secondary"
-				"tall"		"14"
-				"weight"	"400"
-				"additive"	"0"
-				"antialias" "1"
-			}
-		}
-		"MatchSummaryWinner"
-		{
-			"1"
-			{
-				"name"		"TF2 Secondary"
-				"tall"		"20"
-				"weight"	"400"
-				"additive"	"0"
-				"antialias" "1"
-			}
-		}
-		"CompMatchStartTeamNames"
-		{
-			"1"
-			{
-				"name"		"TF2 Secondary"
-				"tall"		"14"
-				"weight"	"400"
-				"additive"	"0"
-				"antialias" "1"
-			}
-		}
+		
 		"ControlPointTimer"
 		{
 			"1"
@@ -2815,8 +4736,6 @@ Scheme
 			{
 				"name"		"TF2"
 				"tall"		"18"
-				"tall_hidef"	"24"
-				"tall_lodef"	"18"
 				"weight"	"500"
 				"additive"	"0"
 				"antialias" "1"
@@ -2839,8 +4758,6 @@ Scheme
 			{
 				"name"		"TF2 Secondary"
 				"tall"		"18"
-				"tall_hidef"	"24"
-				"tall_lodef"	"20"
 				"weight"	"500"
 				"additive"	"0"
 				"antialias" "1"
@@ -2872,7 +4789,7 @@ Scheme
 				"antialias" "1"
 			}
 		}
-
+		
 		ControllerHintText
 		{
 			"1"
@@ -2885,19 +4802,6 @@ Scheme
 				"additive"	"0"
 			}
 		}
-
-		"MMenuPlayListDesc"
-		{
-			"1"
-			{
-				"name"		"TF2 Secondary"
-				"tall"		"9"
-				"weight"	"400"
-				"additive"	"0"
-				"antialias" 	"1"
-			}
-		}
-
 	}
 
 	//
@@ -3239,8 +5143,8 @@ Scheme
 			"image"					"../hud/color_panel_brown"
 			"src_corner_height"		"23"				// pixels inside the image
 			"src_corner_width"		"23"
-			"draw_corner_width"		"5"				// screen size of the corners ( and sides ), proportional
-			"draw_corner_height" 	"5"	
+			"draw_corner_width"		"3"				// screen size of the corners ( and sides ), proportional
+			"draw_corner_height" 	"3"	
 		}
 		TFFatLineBorderRedBGOpaque
 		{
@@ -3272,19 +5176,8 @@ Scheme
 			"image"					"../hud/color_panel_red"
 			"src_corner_height"		"23"				// pixels inside the image
 			"src_corner_width"		"23"
-			"draw_corner_width"		"5"				// screen size of the corners ( and sides ), proportional
-			"draw_corner_height" 	"5"	
-		}
-		TFFatLineBorderRedBGMoreOpaque
-		{
-			"bordertype"			"scalable_image"
-			"backgroundtype"		"2"
-			
-			"image"					"../hud/color_panel_red_more_opaque"
-			"src_corner_height"		"23"				// pixels inside the image
-			"src_corner_width"		"23"
-			"draw_corner_width"		"5"				// screen size of the corners ( and sides ), proportional
-			"draw_corner_height" 	"5"	
+			"draw_corner_width"		"3"				// screen size of the corners ( and sides ), proportional
+			"draw_corner_height" 	"3"	
 		}
 		TFFatLineBorderBlueBG
 		{
@@ -3294,41 +5187,8 @@ Scheme
 			"image"					"../hud/color_panel_blu"
 			"src_corner_height"		"23"				// pixels inside the image
 			"src_corner_width"		"23"
-			"draw_corner_width"		"5"				// screen size of the corners ( and sides ), proportional
-			"draw_corner_height" 	"5"	
-		}
-		TFFatLineBorderBlueBGMoreOpaque
-		{
-			"bordertype"			"scalable_image"
-			"backgroundtype"		"2"
-			
-			"image"					"../hud/color_panel_blu_more_opaque"
-			"src_corner_height"		"23"				// pixels inside the image
-			"src_corner_width"		"23"
-			"draw_corner_width"		"5"				// screen size of the corners ( and sides ), proportional
-			"draw_corner_height" 	"5"	
-		}
-		TFFatLineBorderBlueBGOpaque
-		{
-			"bordertype"			"scalable_image"
-			"backgroundtype"		"2"
-			
-			"image"					"../hud/color_panel_blu_opaque"
-			"src_corner_height"		"23"				// pixels inside the image
-			"src_corner_width"		"23"
-			"draw_corner_width"		"5"				// screen size of the corners ( and sides ), proportional
-			"draw_corner_height" 	"5"	
-		}
-		TFFatLineBorderClearBG
-		{
-			"bordertype"			"scalable_image"
-			"backgroundtype"		"2"
-			
-			"image"					"../hud/color_panel_clear"
-			"src_corner_height"		"23"				// pixels inside the image
-			"src_corner_width"		"23"
-			"draw_corner_width"		"5"				// screen size of the corners ( and sides ), proportional
-			"draw_corner_height" 	"5"	
+			"draw_corner_width"		"3"				// screen size of the corners ( and sides ), proportional
+			"draw_corner_height" 	"3"	
 		}
 
 		ToolTipBorder
@@ -3589,7 +5449,7 @@ Scheme
 			"draw_corner_width"		"5"				// screen size of the corners ( and sides ), proportional
 			"draw_corner_height" 	"5"	
 		}
-
+		
 		// Rarity1 ------------------------------------------------------------------------------------------------
 		BackpackItemBorder_1
 		{
@@ -3639,7 +5499,7 @@ Scheme
 			"draw_corner_width"		"5"				// screen size of the corners ( and sides ), proportional
 			"draw_corner_height" 	"5"	
 		}
-
+		
 		// Rarity2 ------------------------------------------------------------------------------------------------
 		BackpackItemBorder_2
 		{
@@ -3689,7 +5549,7 @@ Scheme
 			"draw_corner_width"		"5"				// screen size of the corners ( and sides ), proportional
 			"draw_corner_height" 	"5"	
 		}
-
+		
 		// Rarity3 ------------------------------------------------------------------------------------------------
 		BackpackItemBorder_3
 		{
@@ -3739,7 +5599,7 @@ Scheme
 			"draw_corner_width"		"5"				// screen size of the corners ( and sides ), proportional
 			"draw_corner_height" 	"5"	
 		}
-
+		
 		// Rarity4 ------------------------------------------------------------------------------------------------
 		BackpackItemBorder_4
 		{
@@ -3789,7 +5649,7 @@ Scheme
 			"draw_corner_width"		"5"				// screen size of the corners ( and sides ), proportional
 			"draw_corner_height" 	"5"	
 		}
-
+		
 		// Haunted ------------------------------------------------------------------------------------------------
 		BackpackItemBorder_Haunted
 		{
@@ -3839,7 +5699,7 @@ Scheme
 			"draw_corner_width"		"5"				// screen size of the corners ( and sides ), proportional
 			"draw_corner_height" 	"5"	
 		}
-
+		
 		// PaintkitWeapon ------------------------------------------------------------------------------------------------
 		BackpackItemBorder_PaintkitWeapon
 		{
@@ -3939,7 +5799,7 @@ Scheme
 			"draw_corner_width"		"5"				// screen size of the corners ( and sides ), proportional
 			"draw_corner_height" 	"5"	
 		}
-
+		
 		// Vintage ------------------------------------------------------------------------------------------------
 		BackpackItemBorder_Vintage
 		{
@@ -3989,7 +5849,7 @@ Scheme
 			"draw_corner_width"		"5"				// screen size of the corners ( and sides ), proportional
 			"draw_corner_height" 	"5"	
 		}
-
+		
 		// Community ------------------------------------------------------------------------------------------------
 		BackpackItemBorder_Community
 		{
@@ -4039,7 +5899,7 @@ Scheme
 			"draw_corner_width"		"5"				// screen size of the corners ( and sides ), proportional
 			"draw_corner_height" 	"5"	
 		}
-
+		
 		// Developer ------------------------------------------------------------------------------------------------
 		BackpackItemBorder_Developer
 		{
@@ -4089,7 +5949,7 @@ Scheme
 			"draw_corner_width"		"5"				// screen size of the corners ( and sides ), proportional
 			"draw_corner_height" 	"5"	
 		}
-
+		
 		// SelfMade ------------------------------------------------------------------------------------------------
 		BackpackItemBorder_SelfMade
 		{
@@ -4139,7 +5999,7 @@ Scheme
 			"draw_corner_width"		"5"				// screen size of the corners ( and sides ), proportional
 			"draw_corner_height" 	"5"	
 		}
-
+		
 		// Customized ------------------------------------------------------------------------------------------------
 		BackpackItemBorder_Customized
 		{
@@ -4189,7 +6049,7 @@ Scheme
 			"draw_corner_width"		"5"				// screen size of the corners ( and sides ), proportional
 			"draw_corner_height" 	"5"	
 		}
-
+		
 		// Strange ------------------------------------------------------------------------------------------------
 		BackpackItemBorder_Strange
 		{
@@ -4239,7 +6099,7 @@ Scheme
 			"draw_corner_width"		"5"				// screen size of the corners ( and sides ), proportional
 			"draw_corner_height" 	"5"	
 		}
-
+		
 		// Completed ------------------------------------------------------------------------------------------------
 		BackpackItemBorder_Completed
 		{
@@ -4633,7 +6493,7 @@ Scheme
 			"draw_corner_width"		"5"				// screen size of the corners ( and sides ), proportional
 			"draw_corner_height" 	"5"	
 		}
-
+		
 		StoreItemBorder
 		{
 			"inset" "0 0 1 1"
@@ -4754,32 +6614,6 @@ Scheme
 				}
 			}
 		}
-
-		MainMenuButtonGlow
-		{
-			"bordertype"			"scalable_image"
-			"backgroundtype"		"2"
-			"color"					"178 83 22 255"
-			
-			"image"					"button_glow"
-			"src_corner_height"		"4"				// pixels inside the image
-			"src_corner_width"		"4"
-			"draw_corner_width"		"4"				// screen size of the corners ( and sides ), proportional
-			"draw_corner_height" 	"4"	
-		}
-
-		MainMenuButtonGlow2
-		{
-			"bordertype"			"scalable_image"
-			"backgroundtype"		"2"
-			"color"					"238 103 17 255"
-			
-			"image"					"button_glow"
-			"src_corner_height"		"4"				// pixels inside the image
-			"src_corner_width"		"4"
-			"draw_corner_width"		"4"				// screen size of the corners ( and sides ), proportional
-			"draw_corner_height" 	"4"	
-		}
 		
 		NotificationDefault
 		{
@@ -4949,30 +6783,6 @@ Scheme
 			"draw_corner_width"		"4"				// screen size of the corners ( and sides ), proportional
 			"draw_corner_height" 	"4"	
 		}
-
-		StoreHighlightedBorder
-		{
-			"bordertype"			"scalable_image"
-			"backgroundtype"		"2"
-			
-			"image"					"featured_corner"
-			"src_corner_height"		"32"				// pixels inside the image
-			"src_corner_width"		"32"
-			"draw_corner_width"		"4"				// screen size of the corners ( and sides ), proportional
-			"draw_corner_height" 	"4"	
-		}
-
-		StoreHighlightedBackgroundBorder
-		{
-			"bordertype"			"scalable_image"
-			"backgroundtype"		"2"
-			
-			"image"					"store/store_featured_item_bg01"
-			"src_corner_height"		"80"				// pixels inside the image
-			"src_corner_width"		"30"
-			"draw_corner_width"		"0"				// screen size of the corners ( and sides ), proportional
-			"draw_corner_height" 	"0"	
-		}
 		
 		StoreDiscountBorder
 		{
@@ -5104,20 +6914,6 @@ Scheme
 			}
 		}
 
-		SortCategoryBorder
-		{
-			"inset" "0 0 1 1"
-	
-			Right
-			{
-				"1"
-				{
-					"color" "TanDark"
-					"offset" "1 0"
-				}
-			}
-		}
-
 		SteamWorkshopBorder
 		{
 			"inset" "0 0 1 1"
@@ -5163,22 +6959,22 @@ Scheme
 			"bordertype"			"scalable_image"
 			"backgroundtype"		"2"
 			
-			"image"					"replay/fatlineborder_red"
+			"image"					"../hud/color_panel_red"
 			"src_corner_height"		"23"				// pixels inside the image
 			"src_corner_width"		"23"
-			"draw_corner_width"		"5"				// screen size of the corners ( and sides ), proportional
-			"draw_corner_height" 	"5"	
+			"draw_corner_width"		"3"				// screen size of the corners ( and sides ), proportional
+			"draw_corner_height" 	"3"	
 		}
 		ReplayFatLineBorderOpaque
 		{
 			"bordertype"			"scalable_image"
 			"backgroundtype"		"2"
 			
-			"image"					"replay/fatlineborder"
+			"image"					"../hud/color_panel_brown"
 			"src_corner_height"		"23"				// pixels inside the image
 			"src_corner_width"		"23"
-			"draw_corner_width"		"5"				// screen size of the corners ( and sides ), proportional
-			"draw_corner_height" 	"5"	
+			"draw_corner_width"		"3"				// screen size of the corners ( and sides ), proportional
+			"draw_corner_height" 	"3"	
 		}
 		ReplayGrayDialogBorder
 		{
@@ -5273,54 +7069,6 @@ Scheme
 			"draw_corner_width"		"7"				// screen size of the corners ( and sides ), proportional
 			"draw_corner_height" 	"7"	
 		}
-
-		RedWithThinBorder
-		{
-			"bordertype"			"scalable_image"
-			"backgroundtype"		"2"
-			
-			"image"					"../hud/panel_scalable_red"
-			"src_corner_height"		"23"			// pixels inside the image
-			"src_corner_width"		"23"
-			"draw_corner_width"		"7"				// screen size of the corners ( and sides ), proportional
-			"draw_corner_height" 	"7"	
-		}
-
-		BlueWithThinBorder
-		{
-			"bordertype"			"scalable_image"
-			"backgroundtype"		"2"
-			
-			"image"					"../hud/panel_scalable_blue"
-			"src_corner_height"		"23"			// pixels inside the image
-			"src_corner_width"		"23"
-			"draw_corner_width"		"7"				// screen size of the corners ( and sides ), proportional
-			"draw_corner_height" 	"7"	
-		}
-
-		InnerShadowBorder
-		{
-			"bordertype"			"scalable_image"
-			"backgroundtype"		"2"
-			
-			"image"					"inner_shadow_border"
-			"src_corner_height"		"5"				// pixels inside the image
-			"src_corner_width"		"5"
-			"draw_corner_width"		"5"				// screen size of the corners ( and sides ), proportional
-			"draw_corner_height" 	"5"	
-		}
-
-		InnerShadowBorderThin
-		{
-			"bordertype"			"scalable_image"
-			"backgroundtype"		"2"
-			
-			"image"					"inner_shadow_border"
-			"src_corner_height"		"5"				// pixels inside the image
-			"src_corner_width"		"5"
-			"draw_corner_width"		"4"				// screen size of the corners ( and sides ), proportional
-			"draw_corner_height" 	"4"	
-		}
 	}
 
 	//////////////////////// CUSTOM FONT FILES /////////////////////////////
@@ -5346,7 +7094,7 @@ Scheme
 				"range" "0x0000 0xFFFF"
 			}
 		}
-		"4" 
+		"4"
 		{
 			"font" "resource/TF2Secondary.ttf"
 			"name" "TF2 Secondary"
@@ -5359,7 +7107,7 @@ Scheme
 				"range" "0x0000 0xFFFF"
 			}
 		}
-		"5" 
+		"5"
 		{
 			"font" "resource/TF2Professor.ttf"
 			"name" "TF2 Professor"
@@ -5372,7 +7120,7 @@ Scheme
 				"range" "0x0000 0x00FF"
 			}
 		}	
-		"6" 
+		"6"
 		{
 			"font" "resource/TF2Build.ttf"
 			"name" "TF2 Build"
@@ -5388,7 +7136,53 @@ Scheme
 			{
 				"range" "0x0000 0xFFFF"
 			}
-		}			
+		}
 		"7" "resource/ocra.ttf"
+		"8"
+		{
+			"font" "resource/fonts/Surface.otf"
+			"name" "Surface-Medium"
+		}
+		"9"
+		{
+			"font" "resource/fonts/Surface-Sharpened.ttf"
+			"name" "SurfaceSharpened-Medium"
+		}
+		"10"
+		{
+			"font" "resource/fonts/Surface-Numbers.ttf"
+			"name" "SurfaceNumbers-Medium"
+		}
+		"11"
+		{
+			"font" "resource/fonts/Crosshairs.ttf"
+			"name" "broeselhud Crosshairs"
+		}
+		"12"
+		{
+			"font" "resource/fonts/Blank.ttf"
+			"name" "Blank"
+		}
+		"13"
+		{
+			"font" "resource/fonts/Blocks.ttf"
+			"name" "Blocks"
+		}
+		"14"
+		{
+			"font" "resource/fonts/Garm3nFont.ttf"
+			"name" "Garm3nFont"
+		}
+		"15"
+		{
+			"font" "resource/fonts/tf2cross.ttf"
+			"name" "tf2cross"
+		}
+		
+		"16"
+		{
+			"font" "resource/crosshairs.ttf"
+			"name" "Crosshairs"
+		}
 	}
-}
+}
\ No newline at end of file
