diff --git a/scripts/hudanimations_tf.txt.sensitive b/scripts/hudanimations_tf.txt.sensitive
index 05095eb..5eb9bde 100644
--- a/scripts/hudanimations_tf.txt.sensitive
+++ b/scripts/hudanimations_tf.txt.sensitive
@@ -22,8 +22,6 @@
 //			Spline - simple ease in/out curve
 //			Pulse - < freq > over the duration, the value is pulsed (cosine) freq times ending at the dest value (assuming freq is integral)
 //			Flicker - < randomness factor 0.0 to 1.0 > over duration, each frame if random # is less than factor, use end value, otherwise use prev value
-//			Gain - < bias > Lower bias values bias towards 0.5 and higher bias values bias away from it.
-//			Bias - < bias > Lower values bias the curve towards 0 and higher values bias it towards 1.
 //
 //	RunEvent <event name> <start time>
 //		starts another even running at the specified time
@@ -43,8 +41,10 @@
 //
 //  SetString <panel name> <string varname> <stringvalue> <set time>
 
-event LevelInit
+event DamagedPlayer
 {
+	Animate xHairDmg FgColor "255 255 255 30" Linear 0 0.3
+	Animate xHairDmg FgColor "255 255 255 0" Linear 0.3 0.3
 }
 
 event OpenWeaponSelectionMenu
@@ -208,6 +208,12 @@ event HudHealthBonusPulse
 {
 	Animate	PlayerStatusHealthBonusImage 	Alpha		"255"		Linear 0.0 0.2
 	Animate	PlayerStatusHealthBonusImage 	Alpha		"0"			Linear 0.2 0.4
+	
+	Animate HealthValueShadow		Fgcolor		"200 200 255 255"		Linear 0.0 0.2
+	Animate HealthValueShadow		Fgcolor		"0 0 0 255"				Linear 0.2 0.4
+	
+	Animate CrosshairBonus				tall		40			Linear 0.0 0.075
+//	Animate CrosshairBonus				tall		0			Linear 0.125 0.075
 
 	RunEvent HudHealthBonusPulseLoop	0.4
 }
@@ -222,6 +228,8 @@ event HudHealthBonusPulseStop
 {
 	StopEvent HudHealthBonusPulse 0.0
 	StopEvent HudHealthBonusPulseLoop 0.0
+	Animate CrosshairBonus tall 0 Linear 0.0 0.00001
+	Animate HealthValueShadowFlame		Fgcolor		"0 0 0 255"	Linear 0.0 0.0001
 }
 
 //===========================================
@@ -232,6 +240,12 @@ event HudHealthDyingPulse
 	Animate	PlayerStatusHealthBonusImage 	Alpha		"255"		Linear 0.0 0.075
 	Animate	PlayerStatusHealthBonusImage 	Alpha		"0"			Linear 0.125 0.075
 	
+	Animate HealthValueShadow		Fgcolor		"255 0 0 255"		Linear 0.0 0.075
+	Animate HealthValueShadow		Fgcolor		"0 0 0 255"		Linear 0.125 0.075
+	
+	Animate CrosshairDying				tall		40			Linear 0.0 0.075
+	//Animate CrosshairDying				tall		0			Linear 0.125 0.075
+	
 	RunEvent HudHealthDyingPulseLoop	0.25
 }
 
@@ -245,15 +259,23 @@ event HudHealthDyingPulseStop
 {
 	StopEvent HudHealthDyingPulse 0.0
 	StopEvent HudHealthDyingPulseLoop 0.0
+	Animate CrosshairDying tall 0 Linear 0.0 0.00001
+	Animate HealthValueShadowFlame		Fgcolor		"0 0 0 255"		Linear 0.0 0.0001
 }
 
 //===========================================
 
 event HudLowAmmoPulse
 {
-	Animate	HudWeaponLowAmmoImage		Alpha		"255"		Linear 0.0 0.075
-	Animate	HudWeaponLowAmmoImage		Alpha		"0"			Linear 0.125 0.075
+	Animate	AmmoInClip		Fgcolor		"TanLight"		Linear 0.0 0.075
+	Animate	AmmoInClip		Fgcolor		"255 0 0 255"		Linear 0.125 0.075
 	
+	Animate AmmoInReserve		Fgcolor		"TanLight"		Linear 0.0 0.075
+	Animate AmmoInReserve	Fgcolor		"255 0 0 255"		Linear 0.125 0.075
+
+	Animate AmmoNoClip		Fgcolor		"TanLight"		Linear 0.0 0.075
+	Animate AmmoNoClip		Fgcolor		"255 0 0 255"		Linear 0.125 0.075
+
 	RunEvent HudLowAmmoPulseLoop	0.25
 }
 
@@ -265,6 +287,9 @@ event HudLowAmmoPulseLoop
 
 event HudLowAmmoPulseStop
 {
+	Animate	AmmoInClip		Fgcolor		"TanLight"		Linear 0.0 0.001
+	Animate AmmoInReserve		Fgcolor		"TanLight"		Linear 0.0 0.001
+	Animate AmmoNoClip		Fgcolor		"TanLight"		Linear 0.0 0.001
 	StopEvent HudLowAmmoPulse 0.0
 	StopEvent HudLowAmmoPulseLoop 0.0
 }
@@ -340,7 +365,7 @@ event HudSpyDisguiseChanged
 	Animate PlayerStatusSpyOutlineImage		Alpha		"255"			Linear 0.0 0.2
 	
 	Animate PlayerStatusSpyOutlineImage		Position	"c-200 c-200"	Linear 0.0 0.2
-	Animate PlayerStatusSpyOutlineImage		Size		"400 400"		Linear 0.0 0.2
+	Animate PlayerStatusSpyOutlineImage		Size		"0 0"		Linear 0.0 0.2
 
 	RunEvent HudSpyDisguiseHide	0.7
 }
@@ -348,7 +373,7 @@ event HudSpyDisguiseChanged
 event HudSpyDisguiseHide
 {
 	Animate PlayerStatusSpyOutlineImage		Position	"3 413"			Linear 0.0 0.2
-	Animate PlayerStatusSpyOutlineImage		Size		"55 55"			Linear 0.0 0.2
+	Animate PlayerStatusSpyOutlineImage		Size		"0 0"			Linear 0.0 0.2
 	
 	Animate PlayerStatusSpyOutlineImage		Alpha		"0"				Linear 0.2 0.1
 }
@@ -424,13 +449,23 @@ event HudTournamentSetupPanelClose
 
 event HudMedicCharged
 {
-	Animate	ChargeLabel 	FgColor		"TanLight"		Linear 0.0 0.1
-	Animate	ChargeLabel 	FgColor		"TanDarker"		Linear 0.3 0.4
-	
-	Animate	ChargeMeter 	FgColor		"TanLight"		Linear 0.0 0.1
-	Animate	ChargeMeter 	FgColor		"TanDarker"		Linear 0.3 0.4
+	Animate CrosshairUber		tall		40			Linear 0.0 0.2
+
+	Animate	ChargeLabel 		FgColor		"Red"			Linear 0.0 0.2
+	Animate	ChargeLabel 		FgColor		"Orange"		Linear 0.2 0.2
+	Animate	ChargeLabel 		FgColor		"255 255 0 255"		Linear 0.4 0.2
+	Animate	ChargeLabel 		FgColor		"0 255 0 255"		Linear 0.6 0.2
+	Animate	ChargeLabel 		FgColor		"0 0 255 0"		Linear 0.8 0.2
+	Animate	ChargeLabel 		FgColor		"255 0 255 255"		Linear 1.0 0.2
 
-	RunEvent HudMedicChargedLoop	0.6
+	Animate	ChargeMeter 		FgColor		"Red"			Linear 0.0 0.2
+	Animate	ChargeMeter 		FgColor		"Orange"		Linear 0.2 0.2
+	Animate	ChargeMeter 		FgColor		"255 255 0 255"		Linear 0.4 0.2
+	Animate	ChargeMeter 		FgColor		"0 255 0 255"		Linear 0.6 0.2
+	Animate	ChargeMeter 		FgColor		"0 0 255 0"		Linear 0.8 0.2
+	Animate	ChargeMeter 		FgColor		"255 0 255 255"		Linear 1.0 0.2
+
+	RunEvent HudMedicChargedLoop	1.2
 }
 
 // call to loop HudHealthBonusPulse
@@ -444,8 +479,9 @@ event HudMedicChargedStop
 	StopEvent HudMedicCharged 0.0
 	StopEvent HudMedicChargedLoop 0.0
 	
-	Animate	ChargeLabel 	FgColor		"TanLight"		Linear 0.0 0.0001
+	Animate	ChargeLabel 	FgColor		"White"			Linear 0.0 0.0001
 	Animate	ChargeMeter 	FgColor		"TanLight"		Linear 0.0 0.0001
+	Animate CrosshairUber tall 0 Linear 0.0 0.00001
 }
 
 //====================================
@@ -523,45 +559,7 @@ event ActiveTimerBGPulse
 	
 	Animate	ActiveTimerBG 	Alpha		"0"			Linear 0.9 0.1
 	Animate	ActiveTimerBG 	Alpha		"255"		Linear 1.1 0.1
-}
-
-event ActiveTimerHighlight
-{
-	Animate TimePanelValue FgColor	TanLight		Linear 0.1 0.1
-	Animate TimePanelValue FgColor	"142 134 115 255"		Linear 0.3 0.1
-
-	Animate TimePanelValue FgColor	TanLight		Linear 0.5 0.1
-	Animate TimePanelValue FgColor	"142 134 115 255"		Linear 0.7 0.1
-
-	Animate TimePanelValue FgColor	TanLight		Linear 0.9 0.1
-	Animate TimePanelValue FgColor	"142 134 115 255"		Linear 1.1 0.1
-
-	Animate TimePanelValue FgColor	TanLight		Linear 1.3 0.3
-	
-}
 
-event ActiveTimerDim
-{
-	// Slow fade out
-	Animate TimePanelValue FgColor	"142 134 115 255"		Linear 0.0 0.4
-}
-
-//===========================================
-
-event OvertimeLabelPulseRed
-{
-	Animate	OvertimeLabel 	FgColor		TanLight	Linear 0.0 0.3
-	Animate	OvertimeLabel 	FgColor		RedSolid	Linear 0.3 0.3
-
-	RunEvent OvertimeLabelPulseRed 0.6
-}
-
-event SuddenDeathLabelPulseRed
-{
-	Animate	SuddenDeathLabel 	FgColor		TanLight	Linear 0.0 0.3
-	Animate	SuddenDeathLabel 	FgColor		RedSolid			Linear 0.3 0.3
-
-	RunEvent SuddenDeathLabelPulseRed 0.2
 }
 
 //===========================================
@@ -1017,7 +1015,7 @@ event CollectionCrafting_LetterSend
 	Animate LetterFront			xpos	c-320	Deaccel 0 0.3
 	Animate DrawingPanel		xpos	c-320	Deaccel 0 0.3
 
-	RunEventChild BehindItemParticlePanel StartParticle2 0.3
+	RunEventChild BehindItemParticlePanel PlayEnvelopSendParticles 0.3
 	Animate Stamp				xpos 1120	Accel 0.3 0.2
 	Animate LetterFront			xpos 1000	Accel 0.3 0.2
 	Animate DrawingPanel		xpos 1000	Accel 0.3 0.2
@@ -1025,17 +1023,17 @@ event CollectionCrafting_LetterSend
 	FireCommand	0.2 "playsound ui/trade_up_envelope_slide_out.wav"
 }
 
-event StartParticle0
+event PlayNewItemParticles
 {
 	FireCommand 0 "start0"
 }
 
-event StartParticle1
+event PlayCrateSmashParticles
 {
 	FireCommand 0 "start1"
 }
 
-event StartParticle2
+event PlayEnvelopSendParticles
 {
 	FireCommand 0 "start2"
 }
@@ -1055,7 +1053,7 @@ event CollectionCrafting_ItemRecieved
 	FireCommand 0	"playcratesequence1"
 	SetVisible	NewItemPanel					0 0 
 	SetVisible ReturnModel						1 0.05
-	RunEventChild BehindItemParticlePanel StartParticle1 0.2
+	RunEventChild BehindItemParticlePanel PlayCrateSmashParticles 0.2
 
 	FireCommand	0 "playsound ui/trade_up_crate_smash.wav"
 
@@ -1063,7 +1061,7 @@ event CollectionCrafting_ItemRecieved
 	SetVisible CloseButton		  1			1.5
 
 	FireCommand 2.1 "playcratesequence2"
-//	RunEventChild BehindItemParticlePanel PlayNewItemParticles 1.5
+	RunEventChild BehindItemParticlePanel PlayNewItemParticles 1.5
 	
 	FireCommand	1.0 "playsound ../player/taunt_medic_heroic.wav"
 }
@@ -1105,44 +1103,6 @@ event CollectionCrafting_ShowFailure
 	SetVisible CloseButton		1	0
 }
 
-event CollectionCrafting_WaitForItemsOnly
-{
-	// Set Everything Invisible
-	SetVisible ReturnModel			0					0
-	SetVisible	SendEvelopeButton		 0				0 
-	SetVisible ResponseTimeout		0					0
-	SetVisible WaitingForResponse	0					0
-	SetVisible ShowExplanationsButton1			0					0
-	SetVisible ShowExplanationsButton2			0					0
-	SetVisible ShowExplanationsButton2	0			0
-	SetVisible	SendEvelopeButton		0			0 
-	SetVisible CloseButton		0					0
-	
-	SetVisible ApplyStampButton		0					0
-	SetVisible Stamp				0					0
-	Animate Stamp					xpos c140	Accel	0 0
-	Animate DrawingPanel			xpos c0		Accel	0 0
-	Animate DrawingPanel			wide 0		Accel	0 0
-	Animate LetterFront				xpos c0		Accel	0 0
-	Animate LetterFront				wide 0		Accel	0 0
-	Animate LetterBack_Top		xpos c-250	Accel	0 0
-	Animate LetterBack_Top		ypos 400	Accel	0 0
-	Animate LetterBack_Top		wide 500	Accel	0 0
-	Animate LetterBack_Bottom		xpos c-250	Accel	0 0
-	Animate LetterBack_Bottom		ypos 400	Accel	0 0
-	Animate LetterBack_Bottom		wide 500	Accel	0 0
-	Animate LetterBack_Flap			xpos c-250	Accel	0 0
-	Animate LetterBack_Flap			ypos 400	Accel	0 0
-	Animate LetterBack_Flap			wide 500	Accel	0 0
-	Animate LetterBack_Flap			tall 0		Deaccel 0.6 0.4
-	Animate LetterBack_Top			ypos 400	Accel	0 0
-	
-	// Slider BG up
-	Animate BG			Position	"0 34"	Deaccel	0	0.3
-
-	// Fade dimmer down
-	Animate Dimmer Alpha "255" Linear 0 0.4
-}
 //--------------------------------------------------------------------------
 event QuestItem_Identify_Expand
 {
@@ -1176,25 +1136,12 @@ event QuestItem_Complete_Progress
 	Animate Dimmer	xpos 142	Linear 0 5
 }
 
-event QuestItem_Options_Flash
-{
-	Animate OptionsButton FgColor LightOrange	Linear 0	 0.1
-	Animate OptionsButton FgColor TanLight		Linear 0.1	 0.1
-	Animate OptionsButton FgColor LightOrange	Linear 0.2	 0.1
-	Animate OptionsButton FgColor TanLight		Linear 0.3	 0.1
-	Animate OptionsButton FgColor LightOrange	Linear 0.4	 0.1
-	Animate OptionsButton FgColor TanLight		Linear 0.5	 0.1
-}
-
 //--------------------------------------------------------------------------
 event QuestItem_Expand
 {
 	Animate FrontFolderContainer		ypos 240	Gain 0.75 0 0.4
 	RunEventChild IdentifyButtonContainer QuestItem_Identify_Expand 0
 	RunEventChild TurnInContainer QuestItem_Complete_Expand 0
-
-	SetInputEnabled FrontFolderContainer 0 0
-	RunEvent QuestItem_Options_Flash 1
 }
 
 event QuestItem_Collapse
@@ -1202,8 +1149,6 @@ event QuestItem_Collapse
 	Animate FrontFolderContainer		ypos 0	Gain 0.75 0 0.4
 	RunEventChild IdentifyButtonContainer QuestItem_Identify_Collapse 0
 	RunEventChild TurnInContainer QuestItem_Complete_Collapse 0
-
-	SetInputEnabled FrontFolderContainer 1 0
 }
 
 event QuestItem_TurningIn
@@ -1216,65 +1161,6 @@ event QuestItem_Reset
 	Animate FrontFolderContainer		ypos 0	Linear 0 0
 	RunEventChild IdentifyButtonContainer QuestItem_Identify_Collapse 0
 	RunEventChild TurnInContainer QuestItem_Complete_Collapse 0
-
-	SetInputEnabled FrontFolderContainer 1 0
-}
-
-//--------------------------------------------------------------------------
-event QuestItem_Operation2_Expand
-{
-	StopEvent QuestItem_Operation2_Collapse 0
-
-	RunEventChild IdentifyButtonContainer QuestItem_Identify_Expand 0.4
-	RunEventChild TurnInContainer QuestItem_Complete_Expand 0.4
-
-	Animate FrontFolderContainer	xpos -270	Gain 0.75 0.4 0.3
-	Animate BackFolderContainer		xpos -270	Gain 0.75 0.4 0.3
-	Animate FrontFolderContainer	xpos -10	Gain 0.75 0.7 0.3
-	Animate BackFolderContainer		xpos 0	Gain 0.75 0.7 0.3
-	SetVisible FrontFolderContainer 0 0.7
-	SetVisible BackFolderContainer 1 0.7
-
-	SetInputEnabled FrontFolderContainer 0 0
-	RunEvent QuestItem_Options_Flash 1
-}
-
-event QuestItem_Operation2_Collapse
-{
-	StopEvent QuestItem_Operation2_Expand 0
-
-	RunEventChild IdentifyButtonContainer QuestItem_Identify_Collapse 0
-	RunEventChild TurnInContainer QuestItem_Complete_Collapse 0
-
-	Animate FrontFolderContainer	xpos -270	Gain 0.75 0.0 0.3
-	Animate BackFolderContainer		xpos -270	Gain 0.75 0.0 0.3
-	Animate FrontFolderContainer	xpos 0	Gain 0.75 0.3 0.4
-	Animate BackFolderContainer		xpos 0	Gain 0.75 0.3 0.4
-	SetVisible FrontFolderContainer 1 0.3
-	SetVisible BackFolderContainer 0 0.3
-
-	SetInputEnabled FrontFolderContainer 1 0
-}
-
-event QuestItem_Operation2_TurningIn
-{
-	RunEventChild TurnInContainer QuestItem_Complete_Progress 0
-}
-
-event QuestItem_Operation2_Reset
-{
-	StopEvent QuestItem_Operation2_Collapse 0
-	StopEvent QuestItem_Operation2_Expand 0
-
-	RunEventChild IdentifyButtonContainer QuestItem_Identify_Collapse 0
-	RunEventChild TurnInContainer QuestItem_Complete_Collapse 0
-
-	Animate FrontFolderContainer	xpos 0	Linear 0 0
-	Animate BackFolderContainer		xpos 0	Linear 0 0
-	SetVisible FrontFolderContainer 1 0
-	SetVisible BackFolderContainer 0 0
-
-	SetInputEnabled FrontFolderContainer 1 0
 }
 
 //--------------------------------------------------------------------------
@@ -1294,7 +1180,6 @@ event QuestItem_Expand_Halloween
 
 	// Immediately disable the mouse proxy so the inactive hint goes away
 	RunEventChild FrontFolderContainer	QuestItem_DisableFrontMouse 0
-	SetInputEnabled FrontFolderContainer 0 0
 
 	// Slide the sleeve
 	Animate SleeveImage xpos 300 Bias 0.2  0.0 0.4
@@ -1309,7 +1194,7 @@ event QuestItem_Expand_Halloween
 	Animate FrontFolderContainer		ypos 240	Gain 0.75 0.4 0.4
 	Animate QuestPaperContainer			tall 300    Gain 0.75 0.4 0.4
 
-	RunEvent QuestItem_Options_Flash 1
+	SetVisible FrontFolderContainer 0 0.8
 }
 
 event QuestItem_Collapse_Halloween
@@ -1330,7 +1215,8 @@ event QuestItem_Collapse_Halloween
 	Animate ReadyToTurnInStatus xpos 10		Bias 0.8 0.5 0.2
 	Animate InactiveStatus		xpos 10		Bias 0.8 0.5 0.2
 	Animate EncodedStatus		xpos 10		Bias 0.8 0.5 0.2
-	SetInputEnabled FrontFolderContainer 1 0
+
+	SetVisible FrontFolderContainer 1 0
 }
 
 event QuestItem_Reset_Halloween
@@ -1348,7 +1234,6 @@ event QuestItem_Reset_Halloween
 
 	// Put the sleeve back
 	Animate SleeveImage xpos 15	Linear 0 0
-	SetInputEnabled FrontFolderContainer 1 0
 }
 
 //--------------------------------------------------------------------------
@@ -1374,7 +1259,7 @@ event QuestItem_Front_NoneSelected
 	StopEvent QuestItem_Front_Selected 0
 	StopEvent QuestItem_Front_OtherSelected 0
 
-	Animate			MainContainer		ypos 120	Gain 0.75 0 0.4
+	Animate			MainContainer		ypos 180	Gain 0.75 0 0.4
 }
 
 //--------------------------------------------------------------------------
@@ -1400,7 +1285,7 @@ event QuestItem_Back_NoneSelected
 	StopEvent QuestItem_Back_Selected 0
 	StopEvent QuestItem_Back_OtherSelected 0
 
-	Animate			MainContainer		ypos 0		Gain 0.75 0 0.4
+	Animate			MainContainer		ypos 100	Gain 0.75 0 0.4
 }
 
 //--------------------------------------------------------------------------
@@ -1415,402 +1300,4 @@ event QuestItem_Highlight_Off_Halloween
 	Animate GlowImage Alpha 0 Bias 0.8 0 0.2
 }
 
-//--------------------------------------------------------------------------
-event CompetitiveGame_LowerChatWindow
-{
-	Animate HudChat ypos r140 Accel 0 0
-}
-
-event CompetitiveGame_RestoreChatWindow
-{
-	Animate HudChat ypos 275 Accel 0 0
-}
-
-event HudTournament_MoveChatWindow
-{
-	RunEvent CompetitiveGame_LowerChatWindow 0
-	RunEvent CompetitiveGame_RestoreChatWindow 10
-}
-
-//--------------------------------------------------------------------------
-
-event HudTournament_MoveTimerDown
-{
-	Animate CountdownLabel ypos 150 Linear 0 0
-	Animate CountdownLabelShadow ypos 150 Linear 0 0
-
-	SetVisible CountdownBG 0 5
-	SetVisible CountdownLabel 1 0
-	SetVisible CountdownLabelShadow 1 0
-
-	Animate CountdownLabel ypos c-3 Gain 0.75 0 0.4
-	Animate CountdownLabelShadow ypos c0 Gain 0.75 0 0.4
-
-	SetVisible	TournamentConditionLabel 0 2
-
-	SetVisible CountdownLabel 0 3
-	SetVisible CountdownLabelShadow 0 3
-}
-event HudTournament_AnimateTeams_In
-{
-	Animate RedTeamPanel xpos c150 Accel 0 0.25
-	Animate BlueTeamPanel xpos c-300 Accel 0 0.25
-}
-
-event HudTournament_AnimateTeams_Out
-{
-	Animate RedTeamPanel xpos r-5 Accel 0 0.68
-	Animate BlueTeamPanel xpos -155 Accel 0 0.68
-}
-
-event HudMatchStatus_ShowMatchStartDoors
-{
-	SetVisible	MatchDoors	1	0
-	SetVisible	BlueTeamPanel	1	0
-	SetVisible	RedTeamPanel	1	0
-	SetVisible CountdownLabel 1 0
-	SetVisible CountdownLabelShadow 1 0
-	Animate CountdownLabel ypos 150 Linear 0 0
-	Animate CountdownLabelShadow ypos 150 Linear 0 0
-
-	Animate CountdownLabel ypos c-3 Gain 0.75 0 0.4
-	Animate CountdownLabelShadow ypos c-0 Gain 0.75 0 0.4
-
-	SetVisible	HudTournamentBG 0	2
-	RunEventChild MatchDoors PlayDoorCloseAnim 0
-	RunEventChild FrontParticlePanel PlayDoorSlamParticles 0.45
-	RunEventChild MatchDoors PlayDoorOpenAnim 7
-
-	RunEvent HudTournament_AnimateTeams_In 0.25
-	RunEvent HudTournament_AnimateTeams_Out 7.70
-
-	SetVisible MatchDoors	0 9
-	SetVisible CountdownLabel 0 9.9
-	SetVisible CountdownLabelShadow 0 9.9
-
-	RunEvent HudTournament_DoorsCleanup 10
-	
-	SetVisible CountdownBG 1 11
-}
-
-event HudMatchStatus_ShowCountdown
-{
-	SetVisible CountdownLabel 1 0
-	SetVisible CountdownLabelShadow 1 0
-	Animate CountdownLabel ypos 150 Linear 0 0
-	Animate CountdownLabelShadow ypos 150 Linear 0 0
-	Animate CountdownLabel ypos c-3 Gain 0.75 0 0.4
-	Animate CountdownLabelShadow ypos c-0 Gain 0.75 0 0.4
-	SetVisible CountdownLabel 0 9.9
-	SetVisible CountdownLabelShadow 0 9.9
-}
-
-event HudTournament_DoorsCleanup
-{
-	SetVisible	MatchDoors	0	0
-	SetVisible	BlueTeamPanel	0	0
-	SetVisible	RedTeamPanel	0	0
-
-	Animate RedTeamPanel xpos r-5 Accel 0 0
-	Animate BlueTeamPanel xpos -155 Accel 0 0
-}
-
-event PlayDoorSlamParticles
-{
-	FireCommand 0 "start0"
-}
-
-event PlayDoorCloseAnim
-{
-	FireCommand 0 "animation close"
-}
-
-event PlayDoorOpenAnim
-{
-	FireCommand 0 "animation open"
-}
-
-event HudTournament_ShowTimerCompetitive
-{
-	Animate CountdownLabel ypos 150 Linear 0 0
-	Animate CountdownLabelShadow ypos 150 Linear 0 0
-
-	SetVisible CountdownBG  1 0
-	SetVisible CountdownLabel 1 0
-	SetVisible CountdownLabelShadow 1 0
-}
-
-event HudTournament_ShowTimerDefault
-{
-	Animate CountdownLabel ypos 425 Linear 0 0
-	Animate CountdownLabelShadow ypos 425 Linear 0 0
-
-	SetVisible CountdownBG  1 0
-	SetVisible CountdownLabel 1 0
-	SetVisible CountdownLabelShadow 1 0
-}
-
-event HudTournament_HideTimer
-{
-	SetVisible CountdownBG  0 0
-	SetVisible CountdownLabel 0 0
-	SetVisible CountdownLabelShadow 0 0
-}
-
-event CompetitiveGame_HideDisconnectButton
-{
-	Animate DisconnectButton ypos r-100 Accel 0 0
-}
-
-event CompetitiveGame_ShowDisconnectButton
-{
-	Animate DisconnectButton ypos r30 Accel 0 0.4
-}
-
-event HudTournament_ShowRoundSign
-{
-	SetVisible RoundSignModel 1 0
-	RunEventChild RoundSignModel HudTournament_PlaySignAnim 0
-	SetVisible RoundSignModel 0 6
-}
-
-event HudTournament_PlaySignAnim
-{
-	FireCommand 0 "animation intro"
-	FireCommand 5 "animation outro"
-}
-
-event TeamStatus_PlayerDead
-{
-	SetVisible DeathPanel  1 0
-	SetVisible SkullPanel  1 0
-	SetVisible respawntime  1 0
-
-	Animate DeathPanel ypos 19 Accel 0 0.2
-	Animate SkullPanel ypos 20 Accel 0 0.2
-	Animate respawntime ypos 21 Accel 0 0.2
-}
-
-event TeamStatus_PlayerAlive
-{
-	Animate DeathPanel ypos 0 Accel 0 0.2
-	Animate SkullPanel ypos 0 Accel 0 0.2
-	Animate respawntime ypos 0 Accel 0 0.2
-
-	SetVisible DeathPanel  0 0.2
-	SetVisible SkullPanel  0 0.2
-	SetVisible respawntime 0 0.2
-}
-
-event HudMatchStatus_ShowMatchWinDoors
-{
-	SetVisible	MatchDoors	1	0
-	SetVisible	FrontParticlePanel	1	0
-
-	RunEventChild MatchDoors PlayDoorCloseAnim 0
-	RunEventChild FrontParticlePanel PlayDoorSlamParticles 0.45
-	RunEventChild MatchDoors PlayDoorOpenAnim 4.5
-
-	SetVisible MatchDoors	0 7
-	SetVisible	FrontParticlePanel	0	7
-}
-
-event HudMatchStatus_ShowMatchWinDoors_NoOpen
-{
-	SetVisible	MatchDoors	1	0
-	SetVisible	FrontParticlePanel	1	0
-
-	RunEventChild MatchDoors PlayDoorCloseAnim 0
-	RunEventChild FrontParticlePanel PlayDoorSlamParticles 0.45
-}
-
-event HudMatchSummary_SlideInPanels
-{
-	Animate BlueTeamPanel xpos 0 Accel 0.25 0.25
-	Animate RedTeamPanel xpos 0 Accel 0.25 0.25
-}
-
-event MMenu_PlayList_Expand
-{
-	Animate ItemsContainer			ypos 420 Spline 0 0.4
-	Animate EventPromo				ypos 470 Spline 0 0.4
-	Animate SafeMode				ypos 470 Spline 0 0.4
-	Animate VRBGPanel				ypos 633 Spline 0 0.4
-	Animate VRModeButton			ypos 639 Spline 0 0.4
-	Animate ShowPromoCodesButton	ypos 639 Spline 0 0.4
-
-	Animate PlayListContainer Tall 260	 Spline	0.0 0.40
-
-	FireCommand 0 "soundentry Panel.SlideDown"
-}
-
-event MMenu_PlayList_Collapse
-{
-	Animate ItemsContainer			ypos 160 Spline 0 0.4
-	Animate EventPromo				ypos 210 Spline 0 0.4
-	Animate SafeMode				ypos 210 Spline 0 0.4
-	Animate VRBGPanel				ypos 373 Spline	0 0.4
-	Animate VRModeButton			ypos 379 Spline 0 0.4
-	Animate ShowPromoCodesButton	ypos 379 Spline 0 0.4
-
-	Animate PlayListContainer Tall 0 Spline 0 0.4
-
-	FireCommand 0 "soundentry Panel.SlideUp"
-}
-
-event MMenu_PlayList_Collapse_Immediate
-{
-	Animate ItemsContainer			ypos 160 Spline 0 0
-	Animate EventPromo				ypos 210 Spline 0 0
-	Animate VRBGPanel				ypos 373 Spline	0 0
-	Animate VRModeButton			ypos 379 Spline 0 0
-	Animate ShowPromoCodesButton	ypos 379 Spline 0 0
-
-	Animate PlayListContainer Tall 0 Spline 0 0
-}
-
-event MMenu_SafeMode_Blink
-{
-	Animate InfoImage	alpha 255 Linear	0.0 0.1 
-	Animate InfoImage	alpha 0 Linear		0.1 0.1
-
-	Animate InfoImage	alpha 255 Linear	0.2 0.1 
-	Animate InfoImage	alpha 0 Linear		0.3 0.1
-
-	Animate InfoImage	alpha 255 Linear		0.4 0.1
-
-	RunEvent MMenu_SafeMode_Blink 1
-}
-
-event PvPRankModelClicked
-{
-	RunEventChild AboveModelParticlePanel StartParticle1 0
-}
-
-event PvPRankLevelUpXPBar
-{
-	Animate ContinuousProgressBar BgColor "100 255 100 255" Linear 0.0 0.1
-	Animate ContinuousProgressBar BgColor ProgressBackground Linear 0.1 0.3
-
-	Animate LevelLabel FgColor "100 255 100 255"	Linear 0.0 0.1
-	Animate LevelLabel FgColor TanLight				Linear 4.0 0.3 
-}
-
-event PvPRankLevelUpModel
-{
-	RunEventChild BelowModelParticlePanel StartParticle0 1.0
-	RunEventChild AboveModelParticlePanel StartParticle0 1.0
-
-	PlaySound 0 "ui/mm_medal_spin_hit.wav"
-}
-
-event PvPRankLevelDownXPBar
-{
-	Animate ContinuousProgressBar FgColor "255 100 100 255" Linear 0.0 0.1
-	Animate ContinuousProgressBar FgColor CreditsGreen Linear 0.1 0.3
-
-	Animate LevelLabel FgColor "255 100 100 255"	Linear 0.0 0.1
-	Animate LevelLabel FgColor TanLight				Linear 4.0 0.3 
-}
-
-event PvPRankLevelDownModel
-{
-	RunEventChild AboveModelParticlePanel StartParticle2 0.15
-	PlaySound 0 "ui/mm_rank_down_medal_shoot.wav"
-	PlaySound 0.15 "ui/mm_rank_down_medal_shoot_impact.wav"
-}
-
-event CompetitiveGame_ShowPvPRankPanel
-{
-	Animate RankPanel ypos r0 Linear 0 0
-	Animate RankPanel ypos r295 Bias 0.90 0.2 0.3
-	RunEventChild RankPanel PvPRankPanelShowXPChange 1
-}
-
-event PvPRankPanelShowXPChange
-{
-	FireCommand 0 "begin_xp_lerp"
-}
-
-event PyroVsHeavyWar_ShowChooseScene
-{
-	// Reset
-	Animate Background	ypos -250 Linear 0 0
-	Animate Heavy		ypos -180 Linear 0 0
-	Animate Pyro		ypos -180 Linear 0 0
-	Animate Foreground	ypos -100 Linear 0 0
-
-	// Paralax
-	Animate Background	ypos -175	Bias 0.9 0 1
-	Animate Heavy		ypos -60	Bias 0.9 0 1
-	Animate Pyro		ypos -60	Bias 0.9 0 1
-	Animate Foreground	ypos 65		Bias 0.9 0 1
-}
-
-event CasualCategory_Checkbox_Hide
-{
-	Animate Checkbutton xpos 250 Deaccel 0 0.2
-	Animate Title		xpos 115 Deaccel 0 0.2
-	Animate TitleShadow	xpos 116 Deaccel 0 0.2
-
-	//Animate Title		FgColor TanDark Deaccel 0 0.2
-}
-
-event CasualCategory_Checkbox_Show
-{
-	Animate Checkbutton xpos 215 Deaccel 0 0.2
-	Animate Title		xpos 100 Deaccel 0 0.2
-	Animate TitleShadow	xpos 101 Deaccel 0 0.2
-
-	//Animate Title		FgColor TanLight Deaccel 0 0.2
-}
-
-event CasualCategory_Selected
-{
-	Animate Shade BgColor "0 0 0 0" Deaccel 0 0.2
-	Animate Title		FgColor TanLight Deaccel 0 0.2
-}
-
-event CasualCategory_NotSelected
-{
-	Animate Shade BgColor "0 0 0 200" Deaccel 0 0.2
-	Animate Title		FgColor TanDark Deaccel 0 0.2
-}
-
-event MMenu_UpdateButton_StartGlow
-{
-	SetVisible	ViewDetailsGlow	1	0
-	Animate ViewDetailsGlow BgColor "238 103 17 255" Linear 0 0
-	Animate ViewDetailsGlow Alpha 0   Linear 0.0 0.2
-	Animate ViewDetailsGlow Alpha 255 Linear 0.2 0.2
-	Animate ViewDetailsGlow Alpha 0	  Linear 0.4 0.2
-	Animate ViewDetailsGlow Alpha 255 Linear 0.6 0.2
-	Animate ViewDetailsGlow Alpha 0	  Linear 0.8 0.2
-	Animate ViewDetailsGlow Alpha 255 Linear 1.0 0.2
-	Animate ViewDetailsGlow BgColor "178 83 22 255" Linear 1.0 1.0
-
-	SetVisible	ViewWarButtonGlow	1	0
-	Animate ViewWarButtonGlow BgColor "238 103 17 255" Linear 0 0
-	Animate ViewWarButtonGlow Alpha 0   Linear 0.0 0.2
-	Animate ViewWarButtonGlow Alpha 255 Linear 0.2 0.2
-	Animate ViewWarButtonGlow Alpha 0	  Linear 0.4 0.2
-	Animate ViewWarButtonGlow Alpha 255 Linear 0.6 0.2
-	Animate ViewWarButtonGlow Alpha 0	  Linear 0.8 0.2
-	Animate ViewWarButtonGlow Alpha 255 Linear 1.0 0.2
-	Animate ViewWarButtonGlow BgColor "178 83 22 255" Linear 1.0 1.0
-
-	RunEvent MMenu_UpdateButton_SubtleGlow 5
-}
-
-event MMenu_UpdateButton_SubtleGlow
-{
-	Animate ViewDetailsGlow BgColor "238 103 17 255" Linear 0 0.2
-	Animate ViewDetailsGlow BgColor "178 83 22 255" Linear 0.2 1.0
-
-	Animate ViewWarButtonGlow BgColor "238 103 17 255" Linear 0 0.2
-	Animate ViewWarButtonGlow BgColor "178 83 22 255" Linear 0.2 1.0
-
-	RunEvent MMenu_UpdateButton_SubtleGlow 5
-}
-
 //
