diff --git a/scripts/hudlayout.res.sensitive b/scripts/hudlayout.res.sensitive
index c8e7e41..a579a4b 100644
--- a/scripts/hudlayout.res.sensitive
+++ b/scripts/hudlayout.res.sensitive
@@ -1,14 +1,29 @@
 "Resource/HudLayout.res"
 {
+	"Transparent"
+	{
+		"ControlName"	"ImagePanel"
+		"fieldName" 	"Transparent"
+		"xpos"			"0"
+		"ypos"			"0"
+		"zpos"			"-100"
+		"wide"			"f0"
+		"tall"			"480"
+		"visible"		"0" // TVM
+		"enabled"		"0" // TVM
+		"image"			"replay/thumbnails/transparent"
+		"scaleImage"	"1"
+	}
+	
 	HudPlayerStatus
 	{
 		"fieldName" "HudPlayerStatus"
 		"visible" "1"
 		"enabled" "1"
-		"xpos"	"0"
-		"ypos"	"0"
+		"xpos"	"c-215"
+		"ypos"	"-82"
 		"wide"	"f0"
-		"tall"	"480"
+		"tall"	"600"
 	}
 
 	HudWeaponAmmo
@@ -16,16 +31,14 @@
 		"fieldName" "HudWeaponAmmo"
 		"visible" "1"
 		"enabled" "1"
-		"xpos"	"r95"	[$WIN32]
-		"xpos_minmode"	"r85"	[$WIN32]
-		"ypos"	"r55"	[$WIN32]
-		"ypos_minmode"	"r36"	[$WIN32]
-		"xpos"	"r131"	[$X360]
-		"ypos"	"r77"	[$X360]
-		"wide"	"94"
-		"tall"	"45"
+		"xpos"	"c+27"
+		"ypos"	"c+15"
+		"wide"	"f0"
+		"tall"	"480"
+		"tall_minmode"	"480"
 	}
 	
+		
 	HudObjectiveStatus
 	{
 		"fieldName" "HudObjectiveStatus"
@@ -42,31 +55,27 @@
 		"fieldName" "HudKothTimeStatus"
 		"visible" "1"
 		"enabled" "1"
-		"xpos"	"c-100"
+		"xpos"	"c-65"
 		"ypos"	"0"
-		"wide"	"200"
-		"tall"	"160"
+		"wide"	"130"
+		"tall"	"60"
 		
-		"blue_active_xpos"			"16"
-		"blue_active_xpos_minmode"	"55"
+		"blue_active_xpos"			"0"
+		"blue_active_xpos_minmode"	"0"
 	
-		"red_active_xpos"			"106"
-		"red_active_xpos_minmode"	"105"
-	}	
+		"red_active_xpos"			"70"
+		"red_active_xpos_minmode"	"70"
+	}
 
 	HudItemEffectMeter
 	{
 		"fieldName"		"HudItemEffectMeter"
 		"visible"		"1"
 		"enabled"		"1"
-		"xpos"			"r162"	[$WIN32]
-		"xpos_minmode"	"r52"	[$WIN32]
-		"ypos"			"r52"	[$WIN32]
-		"ypos_minmode"	"r50"	[$WIN32]
-		"xpos"			"r194"	[$X360]
-		"ypos"			"r74"	[$X360]
-		"wide"			"100"
-		"tall"			"50"
+		"xpos"			"c-75"	[$WIN32]
+		"ypos"			"c25"	[$WIN32]
+		"wide"			"500"
+		"tall"			"500"
 		"MeterFG"		"White"
 		"MeterBG"		"Gray"
 	}
@@ -76,10 +85,8 @@
 		"fieldName"		"HudMedicCharge"
 		"visible"		"1"
 		"enabled"		"1"
-		"xpos"			"r138"	[$WIN32]
-		"xpos_minmode"	"r100"	[$WIN32]
-		"ypos"			"r69"	[$WIN32]
-		"ypos_minmode"	"r34"	[$WIN32]
+		"xpos"			"c50"	[$WIN32]
+		"ypos"			"c82"	[$WIN32]
 		"xpos"			"r174"	[$X360]
 		"ypos"			"r90"	[$X360]
 		"wide"			"200"
@@ -91,35 +98,30 @@
 		"fieldName"		"HudDemomanCharge"
 		"visible"		"1"
 		"enabled"		"1"
-		"xpos"			"r80"	[$WIN32]
-		"xpos_minmode"	"r52"	[$WIN32]
-		"ypos"			"r21"	[$WIN32]
-		"ypos_minmode"	"r40"	[$WIN32]
-		"xpos"			"r112"	[$X360]
-		"ypos"			"r45"	[$X360]
+		"xpos"			"c-75"	[$WIN32]
+		"xpos_minmode"	"c-75"	[$WIN32]
+		"ypos"			"c115"	[$WIN32]
+		"ypos_minmode"	"c115"	[$WIN32]
+		"xpos"			"c"	[$X360]
+		"ypos"			"c"	[$X360]
 		"zpos"			"1"			// draw in front of ammo
-		"wide"			"60"
-		"wide_minmode"	"50"
+		"wide"			"300"
+		"wide_minmode"	"300"
 		"tall"			"8"
 		"MeterFG"		"White"
 		"MeterBG"		"Gray"
-	}	
+	}		
 
 	HudBowCharge
 	{
 		"fieldName"		"HudBowCharge"
 		"visible"		"1"
 		"enabled"		"1"
-		"xpos"			"r80"	[$WIN32]
-		"xpos_minmode"	"r52"	[$WIN32]
-		"ypos"			"r21"	[$WIN32]
-		"ypos_minmode"	"r40"	[$WIN32]
-		"xpos"			"r112"	[$X360]
-		"ypos"			"r45"	[$X360]
-		"zpos"			"1"			// draw in front of ammo
-		"wide"			"60"
-		"wide_minmode"	"50"
-		"tall"			"8"
+		"xpos"			"c-100"	
+		"ypos"			"c111"	
+		"zpos"			"1"			// draw in front 										of ammo
+		"wide"			"500"
+		"tall"			"500"
 		"MeterFG"		"White"
 		"MeterBG"		"Gray"
 	}
@@ -129,14 +131,14 @@
 		"fieldName"		"HudFlameRocketCharge"
 		"visible"		"1"
 		"enabled"		"1"
-		"xpos"			"r80"	[$WIN32]
+		"xpos"			"r52"	[$WIN32]
 		"xpos_minmode"	"r52"	[$WIN32]
-		"ypos"			"r21"	[$WIN32]
+		"ypos"			"r40"	[$WIN32]
 		"ypos_minmode"	"r40"	[$WIN32]
 		"xpos"			"r112"	[$X360]
 		"ypos"			"r45"	[$X360]
-		"zpos"			"1"			// draw in front of ammo
-		"wide"			"60"
+		"zpos"			"1"			// draw in front 										of ammo
+		"wide"			"50"
 		"wide_minmode"	"50"
 		"tall"			"8"
 		"MeterFG"		"White"
@@ -150,7 +152,7 @@
 		"enabled"		"1"
 		"xpos"			"c-100"
 		"ypos"			"42"
-		"zpos"			"0"			// draw in front of ammo
+		"zpos"			"0"			// draw in front 										of ammo
 		"wide"			"200"
 		"tall"			"50"
 		"MeterFG"		"Red"
@@ -199,12 +201,9 @@
 	CHudAccountPanel
 	{
 		"fieldName"				"CHudAccountPanel"
-		"xpos"					"r162"	[$WIN32]
-		"ypos"					"r152"	[$WIN32]
-		"ypos_minmode"			"r134"	[$WIN32]
-		"xpos"					"r194"	[$X360]
-		"ypos"					"r174"	[$X360]
-		"wide"					"116"
+		"xpos"					"c-80"
+		"ypos"					"320"
+		"wide"					"160"
 		"tall"  				"180"
 		"visible" 				"1"
 		"enabled" 				"1"
@@ -214,12 +213,10 @@
 	CHealthAccountPanel
 	{
 		"fieldName"				"CHealthAccountPanel"
-		"xpos"					"76"
-		"xpos_minmode"			"61"
-		"ypos"					"r152"
-		"ypos_minmode"			"r134"
-		"wide"					"116"
-		"tall"  				"180"
+		"xpos"					"c-135"
+		"ypos"					"c+20"
+		"wide"					"80"
+		"tall"  				"60"
 		"visible" 				"1"
 		"enabled" 				"1"
 		"PaintBackgroundType"	"2"
@@ -258,9 +255,9 @@
 		"visible" 	"0"
 		"enabled" 	"1"
 		"xpos"		"c-126"
-		"ypos"		"285"
+		"ypos"		"280"
 		"wide"	 	"252"
-		"tall"	 	"35"
+		"tall"	 	"28"
 		"tall_minmode"	 	"28"
 		"priority"	"40"
 		"priority_lodef"	"5"
@@ -278,9 +275,9 @@
 		"visible" 	"0"
 		"enabled" 	"1"
 		"xpos"		"c-126"
-		"ypos"		"250"
+		"ypos"		"370"
 		"wide"	 	"252"
-		"tall"	 	"35"
+		"tall"	 	"28"
 		"tall_minmode"	 	"28"
 		"priority"	"40"
 		"priority_lodef" "35"
@@ -295,20 +292,14 @@
 		"visible" 	"0"
 		"enabled" 	"1"
 		"xpos"		"c-126"
-		"ypos"		"335"
+		"ypos"		"315"
 		"wide"	 	"252"
-		"tall"	 	"35"
+		"tall"	 	"28"
 		"tall_minmode"	 	"28"
 		"priority"	"35"
-
-		if_vr
-		{
-			"ypos"		"330"
-			"x_offset"	"20"
-		}
-
 	}
 	
+	
 	BuildingStatus_Spy
 	{
 		"fieldName" "BuildingStatus_Spy"
@@ -478,6 +469,26 @@
 		"wide"	 "640"
 		"tall"	 "480"
 	}
+	
+	xHairDmg
+	{
+		"controlName" "CExLabel"
+		"fieldName" "xHairDmg"
+		"visible" "1"
+		"enabled" "1"
+		"zpos" "2"
+
+		"xpos" "c-101"
+		"ypos" "c-97"
+		"wide" "202"
+		"tall" "198"
+
+		"font" "xHairDmg"
+		"labelText" "o"
+		"textAlignment" "center"
+
+		"fgcolor" "0 0 0 0"
+	}
 
 	HudDeathNotice
 	{
@@ -485,13 +496,13 @@
 		"visible" "1"
 		"enabled" "1"
 		"xpos"	 "r640"	[$WIN32]
-		"ypos"	 "18"	[$WIN32]
+		"ypos"	 "12"	[$WIN32]
 		"xpos"	 "r672"	[$X360]
 		"ypos"	 "35"	[$X360]
 		"wide"	 "628"
 		"tall"	 "468"
 
-		"MaxDeathNotices" "4"
+		"MaxDeathNotices" "6"
 		"IconScale"	  "0.35"
 		"LineHeight"	  "16"
 		"LineSpacing"	  "4"
@@ -600,18 +611,18 @@
 		"fieldName" "HudCloseCaption"
 		"visible"	"1"
 		"enabled"	"1"
-		"xpos"		"c-250"
-		"ypos"		"276"	[$WIN32]
+		"xpos"		"5" [$WIN32]
+		"ypos"		"330"	[$WIN32]
 		"ypos"		"236"	[$X360]
-		"wide"		"500"
-		"tall"		"136"	[$WIN32]
+		"wide"		"220"
+		"tall"		"100"	[$WIN32]
 		"tall"		"176"	[$X360]
 
 		"BgAlpha"	"128"
 
 		"GrowTime"		"0.25"
-		"ItemHiddenTime"	"0.2"  // Nearly same as grow time so that the item doesn't start to show until growth is finished
-		"ItemFadeInTime"	"0.15"	// Once ItemHiddenTime is finished, takes this much longer to fade in
+		"ItemHiddenTime"	"0.2"  // Nearly same as grow time so 								that the item doesn't start 							to show until growth is 								finished
+		"ItemFadeInTime"	"0.15"	// Once ItemHiddenTime is 									finished, takes this 									much longer to fade 									in
 		"ItemFadeOutTime"	"0.3"
 		"topoffset"		"0"
 	}
@@ -619,7 +630,7 @@
 	HudHistoryResource 
 	{
 		"fieldName" "HudHistoryResource"
-		"visible" "0"
+		"visible" "0"		
 		"enabled" "0"
 		"xpos"	 "r640"	[$WIN32]
 		"xpos"	 "r672"	[$X360]
@@ -726,43 +737,35 @@
 		"fieldName" "HudVoiceSelfStatus"
 		"visible" "1"
 		"enabled" "1"
-		"xpos" "r42" 	[$WIN32]
-		"ypos" "355"	[$WIN32]
-		"xpos" "r75"	[$X360]
-		"ypos" "375"	[$X360]
+		"xpos" "r42"
+		"ypos" "343"
 		"wide" "32"
 		"tall" "32"
 	}
 
 	HudVoiceStatus
 	{
-		"fieldName" "HudVoiceStatus"
-		"visible" "1"
-		"enabled" "1"
-		"xpos" "r145" [$WIN32]
-		"ypos" "0" [$WIN32]
-		"xpos" "r210" [$X360]
-		"ypos" "0" [$X360]
-		"wide" "145"
-		"tall" "400"
-
-		"item_wide"	"135"
-		
-		"show_avatar"		"0"
+		"fieldName"				"HudVoiceStatus"
+		"visible"				"1"
+		"enabled"				"1"
+		"xpos"					"r130"
+		"ypos"					"0"
+		"wide"					"150"
+		"tall"					"400"
+
+		"item_wide"				"120"
+		"item_tall"				"28"
 		
-		"show_dead_icon"	"1"
-		"dead_xpos"			"1"
-		"dead_ypos"			"0"
-		"dead_wide"			"16"
-		"dead_tall"			"16"
+		"show_avatar"			"1"
+		"avatar_xpos"			"-15"
+		"avatar_ypos"			"2"
+		"avatar_wide"			"24"
+		"avatar_tall"			"24"
 		
-		"show_voice_icon"	"1"
-		"icon_ypos"			"0"
-		"icon_xpos"			"15"
-		"icon_tall"			"16"
-		"icon_wide"			"16"
+		"show_dead_icon"		"0"
+		"show_voice_icon"		"0"
 		
-		"text_xpos"			"33"
+		"text_xpos"				"30"
 	}
 
 	HudHintDisplay
@@ -828,10 +831,10 @@
 		"tall"					"200"
 		"visible"				"1"
 		"enabled"				"1"
+		
 		"separator_width"		"9"	// distance between the icons (including their backgrounds)
 		"separator_height"		"7"
-		"height_offset"			"0"	[$WIN32] // distance from the bottom of the panel
-		"height_offset"			"26"	[$X360]
+		"height_offset"			"0" // distance from the bottom of the panel
 	}
 
 	HudCapturePanel
@@ -865,8 +868,6 @@
 		"ypos"					"215"
 		"wide"					"300"
 		"tall"					"300"
-		"zpos"					"4"
-		"proportionaltoparent"	"1"
 	}
 
 	ArenaWinPanel
@@ -885,10 +886,10 @@
 		"fieldName"				"PVEWinPanel"
 		"visible"				"1"
 		"enabled"				"1"
-		"xpos"					"c-150"
-		"ypos"					"255"
-		"wide"					"300"
-		"tall"					"215"
+		"xpos"					"0"
+		"ypos"					"0"
+		"wide"					"f0"
+		"tall"					"f0"
 	}
 
 	StatPanel
@@ -896,14 +897,10 @@
 		"fieldName"				"StatPanel"
 		"visible"				"1"
 		"enabled"				"1"
-		"xpos"					"c-133"
-		"xpos_lodef"			"c-190"
-		"ypos"				"270"
-		"ypos_lodef"			"250"
-		"wide"					"266"
-		"wide_lodef"			"400"
-		"tall"					"120"
-		"tall_lodef"			"190"
+		"xpos"					"c-110"
+		"ypos"					"r180"
+		"wide"					"210"
+		"tall"					"200"
 	}
 	
 	FreezePanel
@@ -1003,8 +1000,7 @@
 		"tall"			"200"
 		"PaintBackgroundType"	"0"
 	}
-
-	"HudEurekaEffectTeleportMenu"
+		"HudEurekaEffectTeleportMenu"
 	{
 		"fieldName" 	"HudEurekaEffectTeleportMenu"
 		"visible" 		"1"
@@ -1015,7 +1011,7 @@
 		"tall"			"195"
 		"PaintBackgroundType"	"0"
 	}
-	
+
 	"HudMenuSpyDisguise"
 	{
 		"fieldName" 	"HudMenuSpyDisguise"
@@ -1036,13 +1032,13 @@
 		"fieldName"		"HudDemomanPipes"
 		"visible"		"1"
 		"enabled"		"1"
-		"xpos"			"r162"	[$WIN32]
-		"xpos_minmode"	"r110"	[$WIN32]
-		"ypos"			"r52"	[$WIN32]
-		"ypos_minmode"	"r30"	[$WIN32]
-		"xpos"			"r194"	[$X360]
-		"ypos"			"r74"	[$X360]
-		"wide"			"100"
+		"xpos"			"c-75"	[$WIN32]
+		"xpos_minmode"	"c-75"	[$WIN32]
+		"ypos"			"c125"	[$WIN32]
+		"ypos_minmode"	"c125"	[$WIN32]
+		"xpos"			"c"	[$X360]
+		"ypos"			"c"	[$X360]
+		"wide"			"150"
 		"tall"			"50"
 	}	
 
@@ -1068,7 +1064,7 @@
 		"ypos"					"15"
 		"ypos_lodef"			"75"
 		"wide"					"320"
-		"tall"					"300"
+		"tall"					"245"
 	}
 
 	HudTeamSwitch
@@ -1151,11 +1147,11 @@
 		"visible"				"1"
 		"enabled"				"1"
 		"xpos"					"c-160"
-		"ypos"					"5"
+		"ypos"					"15"
 		"ypos_minmode"				"15"
 		"ypos_lodef"				"75"
 		"wide"					"125"
-		"tall"					"70"
+		"tall"					"55"
 	}
 	
 	NotificationPanel
@@ -1233,7 +1229,7 @@
 		"NormalY"		"10"
 		"EngineerY"		"170"
 		"zpos"			"2"
-		"wide"			"280"
+		"wide"			"200"
 		"tall"			"280"
 		"visible"		"1"
 		"enabled"		"1"	
@@ -1265,11 +1261,11 @@
 		"fieldName"				"HudInspectPanel"
 		"visible"				"0"
 		"enabled"				"1"
-		"xpos"					"r200"
-		"ypos"					"rs1"
+		"xpos"					"r270"
+		"ypos"					"300"
 		"zpos"					"10"
-		"wide"					"190"
-		"tall"					"f0"
+		"wide"					"270"
+		"tall"					"180"
 	}
 	
 	HudTFCrosshair
@@ -1397,15 +1393,18 @@
 
 	"HudMenuTauntSelection"
 	{
-		"fieldName" 	"HudMenuTauntSelection"
-		"visible" 		"1"
-		"enabled" 		"1"
-		"xpos"			"c-235"
-		"ypos"			"c-52"
-		"ypos_hidef"	"c-42"
-		"ypos_lodef"	"c-42"
-		"wide"			"470"
-		"tall"			"200"
+		"fieldName" 			"HudMenuTauntSelection"
+		"visible" 				"1"
+		"enabled" 				"1"
+		"xpos"					"c-240"
+		"zpos"					"r105"
+		"ypos"					"c-12"
+		"wide"					"463"
+		"tall"					"99"
+		"xpos_minmode"			"c-240"
+		"ypos_minmode"			"r105"
+		"wide_minmode"			"463"
+		"tall_minmode"			"99"
 		"PaintBackgroundType"	"0"
 	}
 
@@ -1443,6 +1442,18 @@
 		"tall"	"f0"
 	}
 
+	"QuestLogContainer"
+	{
+		"ControlName"		"EditablePanel"
+		"fieldName" 		"QuestLogContainer"
+		"visible" 		"1"
+		"enabled" 		"1"
+		"xpos"			"0"
+		"ypos"			"0"
+		"wide"	 		"f0"
+		"tall"	 		"f0"
+	}
+
 	"MatchMakingContainer"
 	{
 		"ControlName"		"EditablePanel"
@@ -1484,9 +1495,20 @@
 		"enabled"				"1"
 		"xpos"					"0"
 		"ypos"					"0"
-		"zpos"					"3"
+		"zpos"					"2"
 		"wide"					"f0"
 		"tall"					"f0"
 	}
-}
 
+	HudTeamStatus
+	{
+		"fieldName"				"HudTeamStatus"
+		"visible"				"1"
+		"enabled"				"1"
+		"xpos"					"0"
+		"ypos"					"0"
+		"zpos"					"2"
+		"wide"					"f0"
+		"tall"					"f0"
+	}
+}
\ No newline at end of file
